2025-02-19 03:51:26,235 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 03:51:32,342 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 03:51:32,708 - INFO - Starting Flet web server on port 8550...
2025-02-19 03:51:32,708 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 03:51:32,709 - INFO - Use color emoji: False
2025-02-19 03:51:32,709 - INFO - Route URL strategy configured: path
2025-02-19 03:51:32,735 - INFO - Starting up Flet App Manager
2025-02-19 03:51:32,736 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 03:51:33,153 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmptec0ffum
2025-02-19 03:51:33,153 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 03:51:33,154 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 03:52:08,802 - INFO - New FletApp: 5khWuNGY
2025-02-19 03:52:08,849 - INFO - New session created (1 total): _qv69CwycNRCE6tDJ_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 03:52:08,850 - INFO - Start session: qv69CwycNRCE6tDJ
2025-02-19 03:52:08,852 - INFO - New FletApp: jNM9UWNo
2025-02-19 03:52:08,856 - INFO - Переход на страницу: /login
2025-02-19 03:52:08,858 - INFO - New session created (2 total): _d3uYqXEJi8mMS8me_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:52:08,858 - INFO - Start session: d3uYqXEJi8mMS8me
2025-02-19 03:52:08,865 - INFO - Переход на страницу: /login
2025-02-19 03:52:13,160 - INFO - Session disconnected: _d3uYqXEJi8mMS8me_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:52:15,597 - INFO - New FletApp: mpGEvqQ2
2025-02-19 03:52:15,601 - INFO - Existing session requested: d3uYqXEJi8mMS8me
2025-02-19 03:52:15,601 - INFO - Session reconnected: _d3uYqXEJi8mMS8me_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:52:15,601 - INFO - Disposing FletApp: jNM9UWNo
2025-02-19 03:52:18,801 - INFO - Session disconnected: _qv69CwycNRCE6tDJ_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 03:52:27,102 - INFO - Переход на страницу: /
2025-02-19 03:52:28,681 - INFO - Переход на страницу: /settings
2025-02-19 03:52:34,339 - INFO - Переход на страницу: /
2025-02-19 03:52:34,940 - INFO - Переход на страницу: /settings
2025-02-19 03:52:35,629 - INFO - Переход на страницу: /login
2025-02-19 03:53:16,278 - INFO - Session disconnected: _d3uYqXEJi8mMS8me_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:53:18,358 - INFO - New FletApp: CvVwpasF
2025-02-19 03:53:18,461 - INFO - Existing session requested: d3uYqXEJi8mMS8me
2025-02-19 03:53:18,461 - INFO - Session reconnected: _d3uYqXEJi8mMS8me_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:53:18,461 - INFO - Disposing FletApp: mpGEvqQ2
2025-02-19 03:53:41,995 - INFO - Переход на страницу: /
2025-02-19 03:53:43,457 - INFO - Переход на страницу: /settings
2025-02-19 03:53:44,239 - INFO - Переход на страницу: /login
2025-02-19 03:56:11,458 - INFO - Session disconnected: _d3uYqXEJi8mMS8me_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:56:11,565 - INFO - Shutting down Flet App Manager
2025-02-19 03:56:11,565 - INFO - Deleting temp dir: C:\Temp\tmptec0ffum
2025-02-19 03:56:11,566 - INFO - Flet app has started...
2025-02-19 03:56:11,566 - INFO - Closing Flet web server...
2025-02-19 03:56:17,054 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 03:56:17,439 - INFO - Starting Flet web server on port 8550...
2025-02-19 03:56:17,440 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 03:56:17,440 - INFO - Use color emoji: False
2025-02-19 03:56:17,440 - INFO - Route URL strategy configured: path
2025-02-19 03:56:17,463 - INFO - Starting up Flet App Manager
2025-02-19 03:56:17,463 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 03:56:17,877 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmp350ffj40
2025-02-19 03:56:17,878 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 03:56:17,878 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 03:56:18,371 - INFO - New FletApp: RN6IyGgZ
2025-02-19 03:56:18,382 - INFO - New session created (1 total): _jldjWXPm1euBvr5Q_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:56:18,383 - INFO - Start session: jldjWXPm1euBvr5Q
2025-02-19 03:56:18,391 - INFO - Переход на страницу: /login
2025-02-19 03:56:19,602 - INFO - New FletApp: bPF1UiTY
2025-02-19 03:56:19,621 - INFO - New session created (2 total): _rfM6VxnTHB70o5ZO_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:56:19,621 - INFO - Start session: rfM6VxnTHB70o5ZO
2025-02-19 03:56:19,752 - INFO - Переход на страницу: /login
2025-02-19 03:56:20,505 - INFO - Session disconnected: _jldjWXPm1euBvr5Q_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:56:21,884 - INFO - Session disconnected: _rfM6VxnTHB70o5ZO_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:56:23,257 - INFO - New FletApp: lhYZZFa8
2025-02-19 03:56:23,321 - INFO - Existing session requested: rfM6VxnTHB70o5ZO
2025-02-19 03:56:23,321 - INFO - Session reconnected: _rfM6VxnTHB70o5ZO_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:56:23,321 - INFO - Disposing FletApp: bPF1UiTY
2025-02-19 03:56:26,442 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("'Page' object has no attribute 'run_async'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\login.py", line 46, in _handle_login
    self.page.run_async(reset_color)
    ^^^^^^^^^^^^^^^^^^^
AttributeError: 'Page' object has no attribute 'run_async'
2025-02-19 03:57:21,537 - INFO - Session disconnected: _rfM6VxnTHB70o5ZO_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:57:21,651 - INFO - Shutting down Flet App Manager
2025-02-19 03:57:21,651 - INFO - Deleting temp dir: C:\Temp\tmp350ffj40
2025-02-19 03:57:21,652 - INFO - Flet app has started...
2025-02-19 03:57:21,652 - INFO - Closing Flet web server...
2025-02-19 03:57:26,287 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 03:57:26,694 - INFO - Starting Flet web server on port 8550...
2025-02-19 03:57:26,694 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 03:57:26,694 - INFO - Use color emoji: False
2025-02-19 03:57:26,694 - INFO - Route URL strategy configured: path
2025-02-19 03:57:26,721 - INFO - Starting up Flet App Manager
2025-02-19 03:57:26,721 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 03:57:27,032 - INFO - New FletApp: 2LHXe4Ra
2025-02-19 03:57:27,059 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmp24i2gpmy
2025-02-19 03:57:27,059 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 03:57:27,059 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 03:57:27,067 - INFO - New session created (1 total): _pEkKk7mxRO9I4LL5_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:57:27,067 - INFO - Start session: pEkKk7mxRO9I4LL5
2025-02-19 03:57:27,095 - INFO - Переход на страницу: /login
2025-02-19 03:57:28,047 - INFO - Session disconnected: _pEkKk7mxRO9I4LL5_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:57:28,052 - INFO - New FletApp: tU7jEScq
2025-02-19 03:57:28,239 - INFO - New session created (2 total): _f6QmlOKAJ5xthLpA_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:57:28,239 - INFO - Start session: f6QmlOKAJ5xthLpA
2025-02-19 03:57:28,286 - INFO - Переход на страницу: /login
2025-02-19 03:57:32,458 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("'Page' object has no attribute 'run_async'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\login.py", line 46, in _handle_login
    self.page.run_async(reset_border)
    ^^^^^^^^^^^^^^^^^^^
AttributeError: 'Page' object has no attribute 'run_async'
2025-02-19 03:58:23,644 - INFO - Session disconnected: _f6QmlOKAJ5xthLpA_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:58:23,753 - INFO - Shutting down Flet App Manager
2025-02-19 03:58:23,753 - INFO - Deleting temp dir: C:\Temp\tmp24i2gpmy
2025-02-19 03:58:23,754 - INFO - Flet app has started...
2025-02-19 03:58:23,754 - INFO - Closing Flet web server...
2025-02-19 03:58:27,749 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 03:58:33,883 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 03:58:34,263 - INFO - Starting Flet web server on port 8550...
2025-02-19 03:58:34,264 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 03:58:34,264 - INFO - Use color emoji: False
2025-02-19 03:58:34,264 - INFO - Route URL strategy configured: path
2025-02-19 03:58:34,287 - INFO - Starting up Flet App Manager
2025-02-19 03:58:34,287 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 03:58:34,603 - INFO - New FletApp: KUm1dSNa
2025-02-19 03:58:34,608 - INFO - New session created (1 total): _514mPdOZPOvU24pb_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:58:34,608 - INFO - Start session: 514mPdOZPOvU24pb
2025-02-19 03:58:34,613 - INFO - Переход на страницу: /login
2025-02-19 03:58:34,619 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmptqfjpjlt
2025-02-19 03:58:34,619 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 03:58:34,619 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 03:58:35,562 - INFO - New FletApp: 2d1rVwC4
2025-02-19 03:58:35,582 - INFO - New session created (2 total): _xzJhNBA04ovC083Y_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:58:35,582 - INFO - Start session: xzJhNBA04ovC083Y
2025-02-19 03:58:35,706 - INFO - Переход на страницу: /login
2025-02-19 03:58:38,471 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("'Page' object has no attribute 'run_async'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\login.py", line 46, in _handle_login
    self.page.run_async(reset_border)
    ^^^^^^^^^^^^^^^^^^^
AttributeError: 'Page' object has no attribute 'run_async'
2025-02-19 03:59:05,508 - INFO - Session disconnected: _xzJhNBA04ovC083Y_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:59:05,508 - INFO - Session disconnected: _514mPdOZPOvU24pb_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:59:05,616 - INFO - Shutting down Flet App Manager
2025-02-19 03:59:05,616 - INFO - Deleting temp dir: C:\Temp\tmptqfjpjlt
2025-02-19 03:59:05,617 - INFO - Flet app has started...
2025-02-19 03:59:05,617 - INFO - Closing Flet web server...
2025-02-19 03:59:11,098 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 03:59:11,456 - INFO - Starting Flet web server on port 8550...
2025-02-19 03:59:11,457 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 03:59:11,458 - INFO - Use color emoji: False
2025-02-19 03:59:11,458 - INFO - Route URL strategy configured: path
2025-02-19 03:59:11,482 - INFO - Starting up Flet App Manager
2025-02-19 03:59:11,482 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 03:59:11,800 - INFO - New FletApp: 0RzoMCMY
2025-02-19 03:59:11,822 - INFO - New session created (1 total): _osiZCOLjEKWHjAqa_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:59:11,822 - INFO - Start session: osiZCOLjEKWHjAqa
2025-02-19 03:59:11,830 - INFO - Переход на страницу: /login
2025-02-19 03:59:11,839 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmpadsyrtwu
2025-02-19 03:59:11,840 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 03:59:11,840 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 03:59:12,067 - INFO - New FletApp: 1k0OkLxd
2025-02-19 03:59:12,104 - INFO - New session created (2 total): _1XFFF7ownCFGavQk_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:59:12,105 - INFO - Start session: 1XFFF7ownCFGavQk
2025-02-19 03:59:12,113 - INFO - Переход на страницу: /login
2025-02-19 03:59:12,796 - INFO - New FletApp: fZH83rVp
2025-02-19 03:59:12,893 - INFO - New session created (3 total): _TC14zE1DXYQqp0Ai_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:59:12,894 - INFO - Start session: TC14zE1DXYQqp0Ai
2025-02-19 03:59:12,923 - INFO - Переход на страницу: /login
2025-02-19 03:59:13,377 - INFO - Session disconnected: _1XFFF7ownCFGavQk_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:59:13,963 - INFO - Session disconnected: _osiZCOLjEKWHjAqa_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:59:17,198 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("'Page' object has no attribute 'run_async'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\login.py", line 47, in _handle_login
    self.page.run_async(reset_border)
    ^^^^^^^^^^^^^^^^^^^
AttributeError: 'Page' object has no attribute 'run_async'
2025-02-19 03:59:44,485 - INFO - Session disconnected: _TC14zE1DXYQqp0Ai_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:59:44,594 - INFO - Shutting down Flet App Manager
2025-02-19 03:59:44,595 - INFO - Deleting temp dir: C:\Temp\tmpadsyrtwu
2025-02-19 03:59:44,596 - INFO - Flet app has started...
2025-02-19 03:59:44,596 - INFO - Closing Flet web server...
2025-02-19 03:59:48,839 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 03:59:53,460 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 03:59:53,827 - INFO - Starting Flet web server on port 8550...
2025-02-19 03:59:53,828 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 03:59:53,828 - INFO - Use color emoji: False
2025-02-19 03:59:53,828 - INFO - Route URL strategy configured: path
2025-02-19 03:59:53,854 - INFO - Starting up Flet App Manager
2025-02-19 03:59:53,854 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 03:59:54,189 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmphkti7yye
2025-02-19 03:59:54,189 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 03:59:54,189 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 03:59:54,852 - INFO - New FletApp: r0WABjdy
2025-02-19 03:59:54,858 - INFO - New session created (1 total): _tIZzVTlN2KIuYD2u_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:59:54,858 - INFO - Start session: tIZzVTlN2KIuYD2u
2025-02-19 03:59:54,863 - INFO - Переход на страницу: /login
2025-02-19 03:59:55,190 - INFO - New FletApp: 4Hvh3WWw
2025-02-19 03:59:55,200 - INFO - New session created (2 total): _6OeFPuAFHW2wTRmd_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 03:59:55,201 - INFO - Start session: 6OeFPuAFHW2wTRmd
2025-02-19 03:59:55,282 - INFO - Переход на страницу: /login
2025-02-19 03:59:55,366 - INFO - Session disconnected: _tIZzVTlN2KIuYD2u_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:00:00,360 - ERROR - Task exception was never retrieved
future: <Task finished name='Task-52' coro=<FletApp.__on_event() done, defined at C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py:113> exception=AttributeError("'Page' object has no attribute 'run_async'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py", line 119, in __on_event
    await session.on_event_async(
        Event(e.eventTarget, e.eventName, e.eventData)
    )
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 863, in on_event_async
    await handler(ce)
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\login.py", line 47, in _handle_login
    self.page.run_async(reset_border)
    ^^^^^^^^^^^^^^^^^^^
AttributeError: 'Page' object has no attribute 'run_async'
2025-02-19 04:00:04,609 - ERROR - Task exception was never retrieved
future: <Task finished name='Task-55' coro=<FletApp.__on_event() done, defined at C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py:113> exception=AttributeError("'Page' object has no attribute 'run_async'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py", line 119, in __on_event
    await session.on_event_async(
        Event(e.eventTarget, e.eventName, e.eventData)
    )
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 863, in on_event_async
    await handler(ce)
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\login.py", line 47, in _handle_login
    self.page.run_async(reset_border)
    ^^^^^^^^^^^^^^^^^^^
AttributeError: 'Page' object has no attribute 'run_async'
2025-02-19 04:01:03,992 - INFO - Session disconnected: _6OeFPuAFHW2wTRmd_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:01:04,099 - INFO - Shutting down Flet App Manager
2025-02-19 04:01:04,100 - INFO - Deleting temp dir: C:\Temp\tmphkti7yye
2025-02-19 04:01:04,101 - INFO - Flet app has started...
2025-02-19 04:01:04,101 - INFO - Closing Flet web server...
2025-02-19 04:01:12,924 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:01:13,300 - INFO - Starting Flet web server on port 8550...
2025-02-19 04:01:13,300 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 04:01:13,301 - INFO - Use color emoji: False
2025-02-19 04:01:13,301 - INFO - Route URL strategy configured: path
2025-02-19 04:01:13,326 - INFO - Starting up Flet App Manager
2025-02-19 04:01:13,327 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 04:01:13,665 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmpnyz1k225
2025-02-19 04:01:13,665 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 04:01:13,665 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:01:14,361 - INFO - New FletApp: P07lY7mB
2025-02-19 04:01:14,370 - INFO - New session created (1 total): _LN9A9iDAXkbRxl9T_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:01:14,371 - INFO - Start session: LN9A9iDAXkbRxl9T
2025-02-19 04:01:14,377 - INFO - Переход на страницу: /login
2025-02-19 04:01:14,657 - INFO - New FletApp: huKnCnrD
2025-02-19 04:01:14,673 - INFO - New session created (2 total): _CvN4Wj0QqhsNexMl_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:01:14,674 - INFO - Start session: CvN4Wj0QqhsNexMl
2025-02-19 04:01:14,784 - INFO - Переход на страницу: /login
2025-02-19 04:01:34,267 - INFO - Session disconnected: _CvN4Wj0QqhsNexMl_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:01:34,267 - INFO - Session disconnected: _LN9A9iDAXkbRxl9T_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:01:34,376 - INFO - Shutting down Flet App Manager
2025-02-19 04:01:34,376 - INFO - Deleting temp dir: C:\Temp\tmpnyz1k225
2025-02-19 04:01:34,377 - INFO - Flet app has started...
2025-02-19 04:01:34,377 - INFO - Closing Flet web server...
2025-02-19 04:02:06,220 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:02:13,086 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:02:13,460 - INFO - Starting Flet web server on port 8550...
2025-02-19 04:02:13,461 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 04:02:13,461 - INFO - Use color emoji: False
2025-02-19 04:02:13,461 - INFO - Route URL strategy configured: path
2025-02-19 04:02:13,485 - INFO - Starting up Flet App Manager
2025-02-19 04:02:13,485 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 04:02:13,708 - INFO - New FletApp: Hw7oakqk
2025-02-19 04:02:13,781 - INFO - New session created (1 total): _a3OIKhmPCfMJ9bqZ_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:02:13,781 - INFO - Start session: a3OIKhmPCfMJ9bqZ
2025-02-19 04:02:13,784 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmpx_uy8vtt
2025-02-19 04:02:13,784 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 04:02:13,784 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:02:13,791 - INFO - Переход на страницу: /login
2025-02-19 04:02:14,834 - INFO - New FletApp: dXd5TcdL
2025-02-19 04:02:14,850 - INFO - New session created (2 total): _ELrlAfUtNf0wnGml_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:02:14,852 - INFO - Start session: ELrlAfUtNf0wnGml
2025-02-19 04:02:14,979 - INFO - Переход на страницу: /login
2025-02-19 04:02:23,687 - INFO - Переход на страницу: /
2025-02-19 04:02:25,498 - INFO - Переход на страницу: /settings
2025-02-19 04:02:26,260 - INFO - Переход на страницу: /login
2025-02-19 04:02:34,578 - INFO - Переход на страницу: /
2025-02-19 04:02:35,266 - INFO - Переход на страницу: /settings
2025-02-19 04:02:35,961 - INFO - Переход на страницу: /login
2025-02-19 04:03:00,141 - INFO - Session disconnected: _a3OIKhmPCfMJ9bqZ_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:05:16,426 - INFO - Session disconnected: _ELrlAfUtNf0wnGml_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:05:16,543 - INFO - Shutting down Flet App Manager
2025-02-19 04:05:16,543 - INFO - Deleting temp dir: C:\Temp\tmpx_uy8vtt
2025-02-19 04:05:16,544 - INFO - Flet app has started...
2025-02-19 04:05:16,544 - INFO - Closing Flet web server...
2025-02-19 04:05:37,173 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:05:41,995 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:05:42,371 - INFO - Starting Flet web server on port 8550...
2025-02-19 04:05:42,372 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 04:05:42,372 - INFO - Use color emoji: False
2025-02-19 04:05:42,372 - INFO - Route URL strategy configured: path
2025-02-19 04:05:42,400 - INFO - Starting up Flet App Manager
2025-02-19 04:05:42,400 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 04:05:42,539 - INFO - New FletApp: L32IW4Yx
2025-02-19 04:05:42,543 - INFO - New session created (1 total): _yTuzWadoRqruPX8b_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:05:42,543 - INFO - Start session: yTuzWadoRqruPX8b
2025-02-19 04:05:42,600 - INFO - Переход на страницу: /login
2025-02-19 04:05:42,741 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmpzmyq15un
2025-02-19 04:05:42,741 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 04:05:42,741 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:05:43,869 - INFO - New FletApp: NNmuCoEI
2025-02-19 04:05:43,886 - INFO - New session created (2 total): _Jy1G4YJIA88tSWHB_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:05:43,886 - INFO - Start session: Jy1G4YJIA88tSWHB
2025-02-19 04:05:43,995 - INFO - Переход на страницу: /login
2025-02-19 04:05:45,584 - INFO - Session disconnected: _Jy1G4YJIA88tSWHB_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:05:46,802 - INFO - New FletApp: EDt6F1hV
2025-02-19 04:05:46,908 - INFO - Existing session requested: Jy1G4YJIA88tSWHB
2025-02-19 04:05:46,908 - INFO - Session reconnected: _Jy1G4YJIA88tSWHB_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:05:46,909 - INFO - Disposing FletApp: NNmuCoEI
2025-02-19 04:05:50,272 - INFO - Session disconnected: _Jy1G4YJIA88tSWHB_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:05:50,272 - INFO - Session disconnected: _yTuzWadoRqruPX8b_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:05:50,378 - INFO - Shutting down Flet App Manager
2025-02-19 04:05:50,379 - INFO - Deleting temp dir: C:\Temp\tmpzmyq15un
2025-02-19 04:05:50,382 - INFO - Flet app has started...
2025-02-19 04:05:50,382 - INFO - Closing Flet web server...
2025-02-19 04:06:35,886 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:06:40,846 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:06:41,212 - INFO - Starting Flet web server on port 8550...
2025-02-19 04:06:41,213 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 04:06:41,213 - INFO - Use color emoji: False
2025-02-19 04:06:41,213 - INFO - Route URL strategy configured: path
2025-02-19 04:06:41,238 - INFO - Starting up Flet App Manager
2025-02-19 04:06:41,238 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 04:06:41,653 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmpfj_4y9l5
2025-02-19 04:06:41,654 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 04:06:41,654 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:06:49,296 - INFO - New FletApp: mZAKh1CF
2025-02-19 04:06:49,310 - INFO - New session created (1 total): _hqWhs4sKoc5qJ5hS_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:06:49,310 - INFO - Start session: hqWhs4sKoc5qJ5hS
2025-02-19 04:06:49,413 - INFO - Переход на страницу: /login
2025-02-19 04:08:26,718 - INFO - Session disconnected: _hqWhs4sKoc5qJ5hS_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:08:27,878 - INFO - New FletApp: Owt2JRgB
2025-02-19 04:08:27,996 - INFO - Existing session requested: hqWhs4sKoc5qJ5hS
2025-02-19 04:08:27,996 - INFO - Session reconnected: _hqWhs4sKoc5qJ5hS_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:08:27,997 - INFO - Disposing FletApp: mZAKh1CF
2025-02-19 04:08:29,119 - INFO - Session disconnected: _hqWhs4sKoc5qJ5hS_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:08:29,225 - INFO - Shutting down Flet App Manager
2025-02-19 04:08:29,225 - INFO - Deleting temp dir: C:\Temp\tmpfj_4y9l5
2025-02-19 04:08:29,226 - INFO - Flet app has started...
2025-02-19 04:08:29,227 - INFO - Closing Flet web server...
2025-02-19 04:08:35,664 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:08:36,153 - INFO - Starting Flet web server on port 8550...
2025-02-19 04:08:36,154 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 04:08:36,154 - INFO - Use color emoji: False
2025-02-19 04:08:36,154 - INFO - Route URL strategy configured: path
2025-02-19 04:08:36,187 - INFO - Starting up Flet App Manager
2025-02-19 04:08:36,188 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 04:08:36,561 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmp9mlypnqi
2025-02-19 04:08:36,562 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 04:08:36,562 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:08:37,809 - INFO - New FletApp: JyBwkvjy
2025-02-19 04:08:37,813 - INFO - New session created (1 total): _6Drbnh1rJ5lBzgyc_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:08:37,814 - INFO - Start session: 6Drbnh1rJ5lBzgyc
2025-02-19 04:08:37,818 - INFO - Переход на страницу: /login
2025-02-19 04:08:39,494 - INFO - Session disconnected: _6Drbnh1rJ5lBzgyc_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:08:41,387 - INFO - New FletApp: ffJfDSYo
2025-02-19 04:08:41,501 - INFO - New session created (2 total): _NJVEi26pG9gMb8m3_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:08:41,501 - INFO - Start session: NJVEi26pG9gMb8m3
2025-02-19 04:08:41,512 - INFO - Переход на страницу: /login
2025-02-19 04:09:22,806 - ERROR - Task exception was never retrieved
future: <Task finished name='Task-139' coro=<FletApp.__on_event() done, defined at C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py:113> exception=TimeoutError("Timeout waiting for invokeMethod clientStorage:set({'key': 'logged_in', 'value': 'true'}) call")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py", line 119, in __on_event
    await session.on_event_async(
        Event(e.eventTarget, e.eventName, e.eventData)
    )
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 863, in on_event_async
    await handler(ce)
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\login.py", line 31, in _handle_login
    self.page.client_storage.set("logged_in", True)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\client_storage.py", line 13, in set
    self.__page._invoke_method(
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^
        "clientStorage:set", {"key": key, "value": jv}, wait_for_result=True
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 1214, in _invoke_method
    raise TimeoutError(
        f"Timeout waiting for invokeMethod {method_name}({arguments}) call"
    )
TimeoutError: Timeout waiting for invokeMethod clientStorage:set({'key': 'logged_in', 'value': 'true'}) call
2025-02-19 04:09:27,810 - ERROR - Task exception was never retrieved
future: <Task finished name='Task-140' coro=<FletApp.__on_event() done, defined at C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py:113> exception=TimeoutError("Timeout waiting for invokeMethod clientStorage:set({'key': 'logged_in', 'value': 'true'}) call")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py", line 119, in __on_event
    await session.on_event_async(
        Event(e.eventTarget, e.eventName, e.eventData)
    )
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 863, in on_event_async
    await handler(ce)
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\login.py", line 31, in _handle_login
    self.page.client_storage.set("logged_in", True)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\client_storage.py", line 13, in set
    self.__page._invoke_method(
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^
        "clientStorage:set", {"key": key, "value": jv}, wait_for_result=True
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 1214, in _invoke_method
    raise TimeoutError(
        f"Timeout waiting for invokeMethod {method_name}({arguments}) call"
    )
TimeoutError: Timeout waiting for invokeMethod clientStorage:set({'key': 'logged_in', 'value': 'true'}) call
2025-02-19 04:09:35,175 - ERROR - Task exception was never retrieved
future: <Task finished name='Task-176' coro=<FletApp.__on_event() done, defined at C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py:113> exception=TimeoutError("Timeout waiting for invokeMethod clientStorage:set({'key': 'logged_in', 'value': 'true'}) call")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py", line 119, in __on_event
    await session.on_event_async(
        Event(e.eventTarget, e.eventName, e.eventData)
    )
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 863, in on_event_async
    await handler(ce)
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\login.py", line 31, in _handle_login
    self.page.client_storage.set("logged_in", True)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\client_storage.py", line 13, in set
    self.__page._invoke_method(
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^
        "clientStorage:set", {"key": key, "value": jv}, wait_for_result=True
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 1214, in _invoke_method
    raise TimeoutError(
        f"Timeout waiting for invokeMethod {method_name}({arguments}) call"
    )
TimeoutError: Timeout waiting for invokeMethod clientStorage:set({'key': 'logged_in', 'value': 'true'}) call
2025-02-19 04:10:09,054 - INFO - Session disconnected: _NJVEi26pG9gMb8m3_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:10:09,163 - INFO - Shutting down Flet App Manager
2025-02-19 04:10:09,164 - INFO - Deleting temp dir: C:\Temp\tmp9mlypnqi
2025-02-19 04:10:09,165 - INFO - Flet app has started...
2025-02-19 04:10:09,165 - INFO - Closing Flet web server...
2025-02-19 04:10:13,410 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:10:20,814 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:10:21,168 - INFO - Starting Flet web server on port 8550...
2025-02-19 04:10:21,169 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 04:10:21,169 - INFO - Use color emoji: False
2025-02-19 04:10:21,169 - INFO - Route URL strategy configured: path
2025-02-19 04:10:21,193 - INFO - Starting up Flet App Manager
2025-02-19 04:10:21,193 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 04:10:21,327 - INFO - New FletApp: eNueDxiV
2025-02-19 04:10:21,332 - INFO - New session created (1 total): _6H9kqviJoaykPSHJ_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:10:21,332 - INFO - Start session: 6H9kqviJoaykPSHJ
2025-02-19 04:10:21,337 - INFO - Переход на страницу: /login
2025-02-19 04:10:21,382 - INFO - Переход на страницу: /
2025-02-19 04:10:21,561 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmpzc7ip4qf
2025-02-19 04:10:21,561 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 04:10:21,561 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:10:22,599 - INFO - New FletApp: agMPnxLa
2025-02-19 04:10:22,640 - INFO - New session created (2 total): _Kk9OHV5oEVULbz4f_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:10:22,640 - INFO - Start session: Kk9OHV5oEVULbz4f
2025-02-19 04:10:22,653 - INFO - Переход на страницу: /login
2025-02-19 04:10:22,671 - INFO - Переход на страницу: /
2025-02-19 04:10:22,673 - INFO - Переход на страницу: /login
2025-02-19 04:10:22,741 - INFO - Переход на страницу: /
2025-02-19 04:10:23,113 - INFO - Session disconnected: _6H9kqviJoaykPSHJ_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:10:23,979 - INFO - Переход на страницу: /settings
2025-02-19 04:10:24,734 - INFO - Переход на страницу: /login
2025-02-19 04:10:34,674 - ERROR - Task exception was never retrieved
future: <Task finished name='Task-69' coro=<FletApp.__on_event() done, defined at C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py:113> exception=TimeoutError("Timeout waiting for invokeMethod clientStorage:set({'key': 'logged_in', 'value': 'true'}) call")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py", line 119, in __on_event
    await session.on_event_async(
        Event(e.eventTarget, e.eventName, e.eventData)
    )
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 863, in on_event_async
    await handler(ce)
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\login.py", line 31, in _handle_login
    self.page.client_storage.set("logged_in", True)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\client_storage.py", line 13, in set
    self.__page._invoke_method(
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^
        "clientStorage:set", {"key": key, "value": jv}, wait_for_result=True
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 1214, in _invoke_method
    raise TimeoutError(
        f"Timeout waiting for invokeMethod {method_name}({arguments}) call"
    )
TimeoutError: Timeout waiting for invokeMethod clientStorage:set({'key': 'logged_in', 'value': 'true'}) call
2025-02-19 04:10:39,679 - ERROR - Task exception was never retrieved
future: <Task finished name='Task-76' coro=<FletApp.__on_event() done, defined at C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py:113> exception=TimeoutError("Timeout waiting for invokeMethod clientStorage:set({'key': 'logged_in', 'value': 'true'}) call")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py", line 119, in __on_event
    await session.on_event_async(
        Event(e.eventTarget, e.eventName, e.eventData)
    )
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 863, in on_event_async
    await handler(ce)
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\login.py", line 31, in _handle_login
    self.page.client_storage.set("logged_in", True)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\client_storage.py", line 13, in set
    self.__page._invoke_method(
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^
        "clientStorage:set", {"key": key, "value": jv}, wait_for_result=True
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 1214, in _invoke_method
    raise TimeoutError(
        f"Timeout waiting for invokeMethod {method_name}({arguments}) call"
    )
TimeoutError: Timeout waiting for invokeMethod clientStorage:set({'key': 'logged_in', 'value': 'true'}) call
2025-02-19 04:10:39,685 - INFO - Session disconnected: _Kk9OHV5oEVULbz4f_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:10:39,685 - INFO - New FletApp: kGQo9GTu
2025-02-19 04:10:39,694 - INFO - Existing session requested: Kk9OHV5oEVULbz4f
2025-02-19 04:10:39,695 - INFO - Session reconnected: _Kk9OHV5oEVULbz4f_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:10:39,695 - INFO - Disposing FletApp: agMPnxLa
2025-02-19 04:10:39,729 - INFO - Session disconnected: _Kk9OHV5oEVULbz4f_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:10:41,011 - INFO - New FletApp: zcWBsB3S
2025-02-19 04:10:41,153 - INFO - Existing session requested: Kk9OHV5oEVULbz4f
2025-02-19 04:10:41,154 - INFO - Session reconnected: _Kk9OHV5oEVULbz4f_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:10:41,154 - INFO - Disposing FletApp: kGQo9GTu
2025-02-19 04:10:46,944 - ERROR - Task exception was never retrieved
future: <Task finished name='Task-100' coro=<FletApp.__on_event() done, defined at C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py:113> exception=TimeoutError("Timeout waiting for invokeMethod clientStorage:set({'key': 'logged_in', 'value': 'true'}) call")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py", line 119, in __on_event
    await session.on_event_async(
        Event(e.eventTarget, e.eventName, e.eventData)
    )
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 863, in on_event_async
    await handler(ce)
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\login.py", line 31, in _handle_login
    self.page.client_storage.set("logged_in", True)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\client_storage.py", line 13, in set
    self.__page._invoke_method(
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^
        "clientStorage:set", {"key": key, "value": jv}, wait_for_result=True
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 1214, in _invoke_method
    raise TimeoutError(
        f"Timeout waiting for invokeMethod {method_name}({arguments}) call"
    )
TimeoutError: Timeout waiting for invokeMethod clientStorage:set({'key': 'logged_in', 'value': 'true'}) call
2025-02-19 04:10:51,958 - ERROR - Task exception was never retrieved
future: <Task finished name='Task-103' coro=<FletApp.__on_event() done, defined at C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py:113> exception=TimeoutError("Timeout waiting for invokeMethod clientStorage:set({'key': 'logged_in', 'value': 'true'}) call")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py", line 119, in __on_event
    await session.on_event_async(
        Event(e.eventTarget, e.eventName, e.eventData)
    )
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 863, in on_event_async
    await handler(ce)
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\login.py", line 31, in _handle_login
    self.page.client_storage.set("logged_in", True)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\client_storage.py", line 13, in set
    self.__page._invoke_method(
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^
        "clientStorage:set", {"key": key, "value": jv}, wait_for_result=True
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 1214, in _invoke_method
    raise TimeoutError(
        f"Timeout waiting for invokeMethod {method_name}({arguments}) call"
    )
TimeoutError: Timeout waiting for invokeMethod clientStorage:set({'key': 'logged_in', 'value': 'true'}) call
2025-02-19 04:10:55,731 - INFO - Session disconnected: _Kk9OHV5oEVULbz4f_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:10:56,737 - INFO - New FletApp: CBhmLaxX
2025-02-19 04:10:56,754 - INFO - Existing session requested: Kk9OHV5oEVULbz4f
2025-02-19 04:10:56,754 - INFO - Session reconnected: _Kk9OHV5oEVULbz4f_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:10:56,754 - INFO - Disposing FletApp: zcWBsB3S
2025-02-19 04:10:58,482 - INFO - Session disconnected: _Kk9OHV5oEVULbz4f_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:10:58,590 - INFO - Shutting down Flet App Manager
2025-02-19 04:10:58,590 - INFO - Deleting temp dir: C:\Temp\tmpzc7ip4qf
2025-02-19 04:10:58,591 - INFO - Flet app has started...
2025-02-19 04:10:58,591 - INFO - Closing Flet web server...
2025-02-19 04:11:05,372 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:11:24,736 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:11:25,102 - INFO - Starting Flet web server on port 8550...
2025-02-19 04:11:25,103 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 04:11:25,103 - INFO - Use color emoji: False
2025-02-19 04:11:25,103 - INFO - Route URL strategy configured: path
2025-02-19 04:11:25,126 - INFO - Starting up Flet App Manager
2025-02-19 04:11:25,127 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 04:11:25,455 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmp8a3w8d14
2025-02-19 04:11:25,455 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 04:11:25,455 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:11:25,978 - INFO - New FletApp: 4VCCz2zm
2025-02-19 04:11:25,986 - INFO - New session created (1 total): _4TVTl9MLHOp7JyIL_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:11:25,986 - INFO - Start session: 4TVTl9MLHOp7JyIL
2025-02-19 04:11:25,991 - INFO - Переход на страницу: /login
2025-02-19 04:11:25,995 - INFO - Переход на страницу: /
2025-02-19 04:11:26,465 - INFO - New FletApp: 5vYxMbge
2025-02-19 04:11:26,562 - INFO - New session created (2 total): _vmUhhdvw2WljfN1z_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:11:26,563 - INFO - Start session: vmUhhdvw2WljfN1z
2025-02-19 04:11:26,594 - INFO - Переход на страницу: /login
2025-02-19 04:11:26,601 - INFO - Переход на страницу: /
2025-02-19 04:11:26,603 - INFO - Переход на страницу: /login
2025-02-19 04:11:26,646 - INFO - Переход на страницу: /
2025-02-19 04:11:26,736 - INFO - Session disconnected: _4TVTl9MLHOp7JyIL_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:11:27,661 - INFO - Переход на страницу: /settings
2025-02-19 04:11:28,229 - INFO - Переход на страницу: /login
2025-02-19 04:11:39,152 - ERROR - Task exception was never retrieved
future: <Task finished name='Task-70' coro=<FletApp.__on_event() done, defined at C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py:113> exception=TimeoutError("Timeout waiting for invokeMethod clientStorage:set({'key': 'logged_in', 'value': 'true'}) call")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\site-packages\flet_web\fastapi\flet_app.py", line 119, in __on_event
    await session.on_event_async(
        Event(e.eventTarget, e.eventName, e.eventData)
    )
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 863, in on_event_async
    await handler(ce)
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\login.py", line 33, in _handle_login
    self.page.client_storage.set("logged_in", True)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\client_storage.py", line 13, in set
    self.__page._invoke_method(
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^
        "clientStorage:set", {"key": key, "value": jv}, wait_for_result=True
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 1214, in _invoke_method
    raise TimeoutError(
        f"Timeout waiting for invokeMethod {method_name}({arguments}) call"
    )
TimeoutError: Timeout waiting for invokeMethod clientStorage:set({'key': 'logged_in', 'value': 'true'}) call
2025-02-19 04:11:53,841 - INFO - Session disconnected: _vmUhhdvw2WljfN1z_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:11:53,950 - INFO - Shutting down Flet App Manager
2025-02-19 04:11:53,950 - INFO - Deleting temp dir: C:\Temp\tmp8a3w8d14
2025-02-19 04:11:53,952 - INFO - Flet app has started...
2025-02-19 04:11:53,952 - INFO - Closing Flet web server...
2025-02-19 04:12:17,550 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:12:23,646 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:12:24,012 - INFO - Starting Flet web server on port 8550...
2025-02-19 04:12:24,013 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 04:12:24,013 - INFO - Use color emoji: False
2025-02-19 04:12:24,013 - INFO - Route URL strategy configured: path
2025-02-19 04:12:24,038 - INFO - Starting up Flet App Manager
2025-02-19 04:12:24,039 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 04:12:24,134 - INFO - New FletApp: NhIhAeTs
2025-02-19 04:12:24,138 - INFO - New session created (1 total): _fqOeQBlNpuT4nj3d_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:12:24,138 - INFO - Start session: fqOeQBlNpuT4nj3d
2025-02-19 04:12:24,143 - INFO - Переход на страницу: /login
2025-02-19 04:12:24,146 - INFO - Переход на страницу: /
2025-02-19 04:12:24,353 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmpslempnx7
2025-02-19 04:12:24,353 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 04:12:24,353 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:12:25,295 - INFO - New FletApp: fuzAzoDR
2025-02-19 04:12:25,448 - INFO - New session created (2 total): _re5sdgTxN3h9BlXQ_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:12:25,449 - INFO - Start session: re5sdgTxN3h9BlXQ
2025-02-19 04:12:25,465 - INFO - Переход на страницу: /login
2025-02-19 04:12:25,487 - INFO - Переход на страницу: /
2025-02-19 04:12:26,007 - INFO - Session disconnected: _fqOeQBlNpuT4nj3d_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:12:26,995 - INFO - Переход на страницу: /settings
2025-02-19 04:12:27,616 - INFO - Переход на страницу: /login
2025-02-19 04:12:33,933 - INFO - Переход на страницу: /
2025-02-19 04:12:34,699 - INFO - Переход на страницу: /settings
2025-02-19 04:12:45,300 - INFO - New FletApp: dLzKpTvr
2025-02-19 04:12:45,368 - INFO - New session created (3 total): _DTVY2tAIuoQlaErI_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 04:12:45,368 - INFO - Start session: DTVY2tAIuoQlaErI
2025-02-19 04:12:45,559 - INFO - Переход на страницу: /login
2025-02-19 04:12:50,801 - INFO - Session disconnected: _DTVY2tAIuoQlaErI_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 04:12:52,040 - INFO - New FletApp: Zyb3rlwB
2025-02-19 04:12:52,192 - INFO - Existing session requested: DTVY2tAIuoQlaErI
2025-02-19 04:12:52,193 - INFO - Session reconnected: _DTVY2tAIuoQlaErI_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 04:12:52,193 - INFO - Disposing FletApp: dLzKpTvr
2025-02-19 04:13:09,945 - INFO - Переход на страницу: /
2025-02-19 04:13:11,286 - INFO - Переход на страницу: /settings
2025-02-19 04:13:16,486 - INFO - Переход на страницу: /login
2025-02-19 04:13:20,019 - INFO - Переход на страницу: /login
2025-02-19 04:13:41,773 - INFO - Session disconnected: _re5sdgTxN3h9BlXQ_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 04:13:41,773 - INFO - Session disconnected: _DTVY2tAIuoQlaErI_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 04:13:41,882 - INFO - Shutting down Flet App Manager
2025-02-19 04:13:41,882 - INFO - Deleting temp dir: C:\Temp\tmpslempnx7
2025-02-19 04:13:41,884 - INFO - Flet app has started...
2025-02-19 04:13:41,884 - INFO - Closing Flet web server...
2025-02-19 04:15:13,022 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:15:13,024 - INFO - Starting up TCP server on localhost:51021
2025-02-19 04:15:13,030 - INFO - Flet app has started...
2025-02-19 04:15:13,034 - INFO - App URL: tcp://localhost:51021
2025-02-19 04:15:13,034 - INFO - Starting Flet View app...
2025-02-19 04:15:13,034 - INFO - Looking for Flet executable at: C:\Program Files\Python313\Lib\site-packages\flet_desktop\app\flet\flet.exe
2025-02-19 04:15:13,035 - INFO - Flet View found in: C:\Program Files\Python313\Lib\site-packages\flet_desktop\app\flet\flet.exe
2025-02-19 04:15:13,297 - INFO - App session started
2025-02-19 04:15:13,301 - INFO - Переход на страницу: /login
2025-02-19 04:15:13,302 - INFO - Переход на страницу: /
2025-02-19 04:15:14,918 - INFO - Переход на страницу: /settings
2025-02-19 04:15:21,578 - INFO - Переход на страницу: /
2025-02-19 04:15:22,258 - INFO - Переход на страницу: /settings
2025-02-19 04:15:23,167 - INFO - Переход на страницу: /login
2025-02-19 04:15:28,946 - INFO - Переход на страницу: /
2025-02-19 04:15:29,505 - INFO - Переход на страницу: /settings
2025-02-19 04:15:35,590 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 04:15:35,592 - INFO - Starting up TCP server on localhost:51029
2025-02-19 04:15:35,598 - INFO - Flet app has started...
2025-02-19 04:15:35,602 - INFO - App URL: tcp://localhost:51029
2025-02-19 04:15:35,602 - INFO - Starting Flet View app...
2025-02-19 04:15:35,602 - INFO - Looking for Flet executable at: C:\Program Files\Python313\Lib\site-packages\flet_desktop\app\flet\flet.exe
2025-02-19 04:15:35,602 - INFO - Flet View found in: C:\Program Files\Python313\Lib\site-packages\flet_desktop\app\flet\flet.exe
2025-02-19 04:15:35,873 - INFO - App session started
2025-02-19 04:15:35,877 - INFO - Переход на страницу: /login
2025-02-19 04:15:35,879 - INFO - Переход на страницу: /
2025-02-19 04:15:38,530 - INFO - Переход на страницу: /settings
2025-02-19 04:15:39,487 - INFO - Переход на страницу: /
2025-02-19 04:15:40,027 - INFO - Переход на страницу: /settings
2025-02-19 04:15:40,379 - INFO - Переход на страницу: /
2025-02-19 04:15:40,849 - INFO - Переход на страницу: /settings
2025-02-19 04:15:43,361 - INFO - Переход на страницу: /
2025-02-19 04:15:44,298 - INFO - Переход на страницу: /settings
2025-02-19 04:15:48,447 - INFO - Переход на страницу: /
2025-02-19 04:15:49,039 - INFO - Переход на страницу: /settings
2025-02-19 04:15:49,398 - INFO - Переход на страницу: /
2025-02-19 04:15:49,726 - INFO - Переход на страницу: /settings
2025-02-19 04:15:49,992 - INFO - Переход на страницу: /
2025-02-19 04:15:50,313 - INFO - Переход на страницу: /settings
2025-02-19 04:15:50,582 - INFO - Переход на страницу: /
2025-02-19 04:15:50,875 - INFO - Переход на страницу: /settings
2025-02-19 04:15:51,102 - INFO - Переход на страницу: /
2025-02-19 04:15:51,343 - INFO - Переход на страницу: /settings
2025-02-19 04:15:51,547 - INFO - Переход на страницу: /
2025-02-19 04:15:51,770 - INFO - Переход на страницу: /settings
2025-02-19 04:15:51,975 - INFO - Переход на страницу: /
2025-02-19 04:15:52,203 - INFO - Переход на страницу: /settings
2025-02-19 04:15:52,411 - INFO - Переход на страницу: /
2025-02-19 04:15:52,657 - INFO - Переход на страницу: /settings
2025-02-19 04:15:52,857 - INFO - Переход на страницу: /
2025-02-19 04:15:53,367 - INFO - Переход на страницу: /settings
2025-02-19 04:15:53,686 - INFO - Переход на страницу: /login
2025-02-19 04:16:00,801 - INFO - Переход на страницу: /
2025-02-19 05:46:52,949 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 05:46:52,950 - INFO - Starting up TCP server on localhost:52979
2025-02-19 05:46:52,956 - INFO - Flet app has started...
2025-02-19 05:46:52,959 - INFO - App URL: tcp://localhost:52979
2025-02-19 05:46:52,959 - INFO - Starting Flet View app...
2025-02-19 05:46:52,960 - INFO - Looking for Flet executable at: C:\Program Files\Python313\Lib\site-packages\flet_desktop\app\flet\flet.exe
2025-02-19 05:46:52,960 - INFO - Flet View found in: C:\Program Files\Python313\Lib\site-packages\flet_desktop\app\flet\flet.exe
2025-02-19 05:46:53,227 - INFO - App session started
2025-02-19 05:46:53,233 - INFO - Переход на страницу: /login
2025-02-19 05:46:53,234 - INFO - Переход на страницу: /
2025-02-19 05:46:57,295 - INFO - Переход на страницу: /settings
2025-02-19 05:46:58,223 - INFO - Переход на страницу: /login
2025-02-19 05:47:03,313 - INFO - Переход на страницу: /
2025-02-19 05:50:50,776 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 05:50:50,777 - INFO - Starting up TCP server on localhost:53107
2025-02-19 05:50:50,783 - INFO - Flet app has started...
2025-02-19 05:50:51,059 - INFO - App session started
2025-02-19 05:50:51,063 - INFO - Переход на страницу: /login
2025-02-19 05:50:51,064 - INFO - Переход на страницу: /
2025-02-19 05:51:04,042 - INFO - Переход на страницу: /settings
2025-02-19 05:51:08,710 - INFO - Переход на страницу: /
2025-02-19 05:51:35,697 - INFO - Переход на страницу: /settings
2025-02-19 05:51:37,448 - INFO - Переход на страницу: /
2025-02-19 05:51:47,273 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 05:51:47,274 - INFO - Starting up TCP server on localhost:53125
2025-02-19 05:51:47,280 - INFO - Flet app has started...
2025-02-19 05:51:47,553 - INFO - App session started
2025-02-19 05:51:47,558 - INFO - Переход на страницу: /login
2025-02-19 05:51:47,559 - INFO - Переход на страницу: /
2025-02-19 05:55:18,937 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 05:55:18,939 - INFO - Starting up TCP server on localhost:53223
2025-02-19 05:55:18,945 - INFO - Flet app has started...
2025-02-19 05:55:19,217 - INFO - App session started
2025-02-19 05:55:19,220 - INFO - Переход на страницу: /login
2025-02-19 05:55:19,222 - INFO - Переход на страницу: /
2025-02-19 05:55:58,688 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 05:55:58,691 - INFO - Starting up TCP server on localhost:53223
2025-02-19 05:55:58,698 - INFO - Flet app has started...
2025-02-19 05:55:58,970 - INFO - App session started
2025-02-19 05:55:58,973 - INFO - Переход на страницу: /login
2025-02-19 05:55:58,974 - INFO - Переход на страницу: /
2025-02-19 06:00:27,933 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:00:27,935 - INFO - Starting up TCP server on localhost:53223
2025-02-19 06:00:27,940 - INFO - Flet app has started...
2025-02-19 06:00:28,145 - INFO - App session started
2025-02-19 06:00:28,148 - INFO - Переход на страницу: /login
2025-02-19 06:00:28,149 - INFO - Переход на страницу: /
2025-02-19 06:02:37,809 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:02:37,812 - INFO - Starting up TCP server on localhost:53223
2025-02-19 06:02:37,818 - INFO - Flet app has started...
2025-02-19 06:02:46,962 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:02:46,964 - INFO - Starting up TCP server on localhost:53494
2025-02-19 06:02:46,969 - INFO - Flet app has started...
2025-02-19 06:02:47,221 - INFO - App session started
2025-02-19 06:02:47,225 - INFO - Переход на страницу: /login
2025-02-19 06:02:47,227 - INFO - Переход на страницу: /
2025-02-19 06:03:23,985 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:03:23,987 - INFO - Starting up TCP server on localhost:53494
2025-02-19 06:03:23,992 - INFO - Flet app has started...
2025-02-19 06:03:40,805 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:03:40,807 - INFO - Starting up TCP server on localhost:53516
2025-02-19 06:03:40,812 - INFO - Flet app has started...
2025-02-19 06:03:41,087 - INFO - App session started
2025-02-19 06:03:41,090 - INFO - Переход на страницу: /login
2025-02-19 06:03:41,093 - INFO - Переход на страницу: /
2025-02-19 06:03:44,573 - INFO - Переход на страницу: /home
2025-02-19 06:03:45,703 - INFO - Переход на страницу: /profile
2025-02-19 06:03:46,896 - INFO - Переход на страницу: /home
2025-02-19 06:03:47,584 - INFO - Переход на страницу: /start
2025-02-19 06:04:47,511 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:04:47,513 - INFO - Starting up TCP server on localhost:53549
2025-02-19 06:04:47,518 - INFO - Flet app has started...
2025-02-19 06:04:47,523 - INFO - App URL: tcp://localhost:53549
2025-02-19 06:04:47,523 - INFO - Starting Flet View app...
2025-02-19 06:04:47,524 - INFO - Looking for Flet executable at: C:\Program Files\Python313\Lib\site-packages\flet_desktop\app\flet\flet.exe
2025-02-19 06:04:47,524 - INFO - Flet View found in: C:\Program Files\Python313\Lib\site-packages\flet_desktop\app\flet\flet.exe
2025-02-19 06:04:47,807 - INFO - App session started
2025-02-19 06:04:47,810 - INFO - Переход на страницу: /login
2025-02-19 06:04:47,812 - INFO - Переход на страницу: /
2025-02-19 06:07:37,359 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:07:37,361 - INFO - Starting up TCP server on localhost:53640
2025-02-19 06:07:37,366 - INFO - Flet app has started...
2025-02-19 06:07:37,371 - INFO - App URL: tcp://localhost:53640
2025-02-19 06:07:37,371 - INFO - Starting Flet View app...
2025-02-19 06:07:37,372 - INFO - Looking for Flet executable at: C:\Program Files\Python313\Lib\site-packages\flet_desktop\app\flet\flet.exe
2025-02-19 06:07:37,372 - INFO - Flet View found in: C:\Program Files\Python313\Lib\site-packages\flet_desktop\app\flet\flet.exe
2025-02-19 06:07:37,650 - INFO - App session started
2025-02-19 06:07:37,654 - INFO - Переход на страницу: /login
2025-02-19 06:07:37,655 - INFO - Переход на страницу: /
2025-02-19 06:07:38,593 - INFO - Переход на страницу: /settings
2025-02-19 06:07:39,355 - INFO - Переход на страницу: /
2025-02-19 06:08:41,519 - INFO - Переход на страницу: /settings
2025-02-19 06:08:42,512 - INFO - Переход на страницу: /
2025-02-19 06:08:42,958 - INFO - Переход на страницу: /settings
2025-02-19 06:08:44,026 - INFO - Переход на страницу: /
2025-02-19 06:09:06,441 - INFO - Переход на страницу: /settings
2025-02-19 06:09:07,241 - INFO - Переход на страницу: /
2025-02-19 06:09:39,290 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:09:39,292 - INFO - Starting up TCP server on localhost:53683
2025-02-19 06:09:39,298 - INFO - Flet app has started...
2025-02-19 06:09:39,303 - INFO - App URL: tcp://localhost:53683
2025-02-19 06:09:39,303 - INFO - Starting Flet View app...
2025-02-19 06:09:39,304 - INFO - Looking for Flet executable at: C:\Program Files\Python313\Lib\site-packages\flet_desktop\app\flet\flet.exe
2025-02-19 06:09:39,304 - INFO - Flet View found in: C:\Program Files\Python313\Lib\site-packages\flet_desktop\app\flet\flet.exe
2025-02-19 06:09:39,577 - INFO - App session started
2025-02-19 06:09:39,582 - INFO - Переход на страницу: /login
2025-02-19 06:09:39,583 - INFO - Переход на страницу: /
2025-02-19 06:15:52,214 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:15:52,216 - INFO - Starting up TCP server on localhost:53847
2025-02-19 06:15:52,223 - INFO - Flet app has started...
2025-02-19 06:15:52,227 - INFO - App URL: tcp://localhost:53847
2025-02-19 06:15:52,227 - INFO - Starting Flet View app...
2025-02-19 06:15:52,228 - INFO - Looking for Flet executable at: C:\Program Files\Python313\Lib\site-packages\flet_desktop\app\flet\flet.exe
2025-02-19 06:15:52,228 - INFO - Flet View found in: C:\Program Files\Python313\Lib\site-packages\flet_desktop\app\flet\flet.exe
2025-02-19 06:15:52,518 - INFO - App session started
2025-02-19 06:15:52,523 - INFO - Переход на страницу: /login
2025-02-19 06:15:52,524 - INFO - Переход на страницу: /
2025-02-19 06:15:55,891 - INFO - Переход на страницу: /profile
2025-02-19 06:15:56,499 - INFO - Переход на страницу: /
2025-02-19 06:15:57,938 - INFO - Переход на страницу: /settings
2025-02-19 06:16:00,398 - INFO - Переход на страницу: /login
2025-02-19 06:16:08,338 - INFO - Переход на страницу: /
2025-02-19 06:16:29,004 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:16:39,309 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:16:39,714 - INFO - Starting Flet web server on port 8550...
2025-02-19 06:16:39,714 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 06:16:39,714 - INFO - Use color emoji: False
2025-02-19 06:16:39,714 - INFO - Route URL strategy configured: path
2025-02-19 06:16:39,743 - INFO - Starting up Flet App Manager
2025-02-19 06:16:39,743 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 06:16:40,175 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmpa23m34s_
2025-02-19 06:16:40,175 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 06:16:40,176 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:16:41,188 - INFO - New FletApp: JduMhD9z
2025-02-19 06:16:41,324 - INFO - New session created (1 total): _l4Vvy0b3qvoEFqZx_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:16:41,324 - INFO - Start session: l4Vvy0b3qvoEFqZx
2025-02-19 06:16:41,465 - INFO - Переход на страницу: /login
2025-02-19 06:16:46,137 - INFO - Переход на страницу: /
2025-02-19 06:16:47,551 - INFO - Переход на страницу: /settings
2025-02-19 06:16:50,690 - INFO - Переход на страницу: /
2025-02-19 06:16:51,629 - INFO - Переход на страницу: /profile
2025-02-19 06:16:53,047 - INFO - Переход на страницу: /
2025-02-19 06:16:53,610 - INFO - Переход на страницу: /profile
2025-02-19 06:17:01,068 - INFO - Переход на страницу: /
2025-02-19 06:17:01,597 - INFO - Переход на страницу: /profile
2025-02-19 06:17:02,036 - INFO - Переход на страницу: /
2025-02-19 06:17:02,371 - INFO - Переход на страницу: /profile
2025-02-19 06:17:02,816 - INFO - Переход на страницу: /
2025-02-19 06:17:03,115 - INFO - Переход на страницу: /profile
2025-02-19 06:17:03,552 - INFO - Переход на страницу: /
2025-02-19 06:17:03,838 - INFO - Переход на страницу: /profile
2025-02-19 06:17:04,105 - INFO - Переход на страницу: /
2025-02-19 06:17:04,362 - INFO - Переход на страницу: /profile
2025-02-19 06:17:04,639 - INFO - Переход на страницу: /
2025-02-19 06:17:04,880 - INFO - Переход на страницу: /profile
2025-02-19 06:17:05,185 - INFO - Переход на страницу: /
2025-02-19 06:17:05,431 - INFO - Переход на страницу: /profile
2025-02-19 06:17:05,801 - INFO - Переход на страницу: /
2025-02-19 06:17:06,055 - INFO - Переход на страницу: /profile
2025-02-19 06:17:13,096 - INFO - Session disconnected: _l4Vvy0b3qvoEFqZx_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:17:15,513 - INFO - New FletApp: awv88a5n
2025-02-19 06:17:15,694 - INFO - New session created (2 total): _XNO2IkVrALqeuAy2_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:17:15,695 - INFO - Start session: XNO2IkVrALqeuAy2
2025-02-19 06:17:15,787 - INFO - Переход на страницу: /login
2025-02-19 06:17:15,877 - INFO - Переход на страницу: /
2025-02-19 06:17:15,877 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("'HomePage' object has no attribute 'selected_index'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 6, in __init__
    super().__init__(page)
    ~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 13, in _build
    self._build_mobile_tablet()
    ~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 99, in _build_mobile_tablet
    selected=self.selected_index == 0,
             ^^^^^^^^^^^^^^^^^^^
AttributeError: 'HomePage' object has no attribute 'selected_index'
2025-02-19 06:17:15,904 - INFO - Переход на страницу: /login
2025-02-19 06:17:16,005 - INFO - Переход на страницу: /
2025-02-19 06:17:16,006 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("'HomePage' object has no attribute 'selected_index'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 6, in __init__
    super().__init__(page)
    ~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 13, in _build
    self._build_mobile_tablet()
    ~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 99, in _build_mobile_tablet
    selected=self.selected_index == 0,
             ^^^^^^^^^^^^^^^^^^^
AttributeError: 'HomePage' object has no attribute 'selected_index'
2025-02-19 06:17:37,042 - INFO - New FletApp: 2Nsydvd5
2025-02-19 06:17:37,448 - INFO - New session created (3 total): _C9EooqRXcznqGv3X_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:17:37,449 - INFO - Start session: C9EooqRXcznqGv3X
2025-02-19 06:17:37,493 - INFO - Переход на страницу: /login
2025-02-19 06:17:43,807 - INFO - Переход на страницу: /
2025-02-19 06:17:43,807 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("'HomePage' object has no attribute 'selected_index'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 6, in __init__
    super().__init__(page)
    ~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 13, in _build
    self._build_mobile_tablet()
    ~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 99, in _build_mobile_tablet
    selected=self.selected_index == 0,
             ^^^^^^^^^^^^^^^^^^^
AttributeError: 'HomePage' object has no attribute 'selected_index'
2025-02-19 06:17:45,241 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("'NoneType' object has no attribute 'client_storage'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\login.py", line 31, in _handle_login
    self.page.client_storage.set("logged_in", True)
    ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'client_storage'
2025-02-19 06:17:50,417 - INFO - Session disconnected: _XNO2IkVrALqeuAy2_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:17:52,414 - INFO - New FletApp: 1PgWMlTS
2025-02-19 06:17:52,589 - INFO - Existing session requested: XNO2IkVrALqeuAy2
2025-02-19 06:17:52,590 - INFO - Session reconnected: _XNO2IkVrALqeuAy2_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:17:52,590 - INFO - Disposing FletApp: awv88a5n
2025-02-19 06:18:37,834 - INFO - New FletApp: hM5W3GvH
2025-02-19 06:18:38,162 - INFO - New session created (4 total): _gdkKD7PYPrOTPcdn_6adfa3f7ac30bf9f7a7dc95e3e9e0f69152d3460
2025-02-19 06:18:38,162 - INFO - Start session: gdkKD7PYPrOTPcdn
2025-02-19 06:18:38,190 - INFO - Переход на страницу: /login
2025-02-19 06:18:38,256 - INFO - Переход на страницу: /
2025-02-19 06:18:38,256 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("'HomePage' object has no attribute 'selected_index'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 6, in __init__
    super().__init__(page)
    ~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 13, in _build
    self._build_mobile_tablet()
    ~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 99, in _build_mobile_tablet
    selected=self.selected_index == 0,
             ^^^^^^^^^^^^^^^^^^^
AttributeError: 'HomePage' object has no attribute 'selected_index'
2025-02-19 06:18:41,906 - INFO - Session disconnected: _gdkKD7PYPrOTPcdn_6adfa3f7ac30bf9f7a7dc95e3e9e0f69152d3460
2025-02-19 06:18:42,363 - INFO - New FletApp: rWfu11pM
2025-02-19 06:18:42,481 - INFO - Existing session requested: gdkKD7PYPrOTPcdn
2025-02-19 06:18:42,482 - INFO - Session reconnected: _gdkKD7PYPrOTPcdn_6adfa3f7ac30bf9f7a7dc95e3e9e0f69152d3460
2025-02-19 06:18:42,482 - INFO - Disposing FletApp: hM5W3GvH
2025-02-19 06:18:53,599 - INFO - Session disconnected: _XNO2IkVrALqeuAy2_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:18:53,599 - INFO - Session disconnected: _gdkKD7PYPrOTPcdn_6adfa3f7ac30bf9f7a7dc95e3e9e0f69152d3460
2025-02-19 06:18:53,599 - INFO - Session disconnected: _C9EooqRXcznqGv3X_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:18:53,705 - INFO - Shutting down Flet App Manager
2025-02-19 06:18:53,705 - INFO - Deleting temp dir: C:\Temp\tmpa23m34s_
2025-02-19 06:18:53,706 - INFO - Flet app has started...
2025-02-19 06:18:53,706 - INFO - Closing Flet web server...
2025-02-19 06:18:59,228 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:18:59,615 - INFO - Starting Flet web server on port 8550...
2025-02-19 06:18:59,615 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 06:18:59,616 - INFO - Use color emoji: False
2025-02-19 06:18:59,616 - INFO - Route URL strategy configured: path
2025-02-19 06:18:59,642 - INFO - Starting up Flet App Manager
2025-02-19 06:18:59,643 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 06:18:59,850 - INFO - New FletApp: dPXW6ZG9
2025-02-19 06:18:59,998 - INFO - New session created (1 total): _vv4dtiqdme4k4kH4_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:18:59,998 - INFO - Start session: vv4dtiqdme4k4kH4
2025-02-19 06:19:00,006 - INFO - Переход на страницу: /login
2025-02-19 06:19:00,012 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmpq1hdgkuk
2025-02-19 06:19:00,012 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 06:19:00,012 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:19:00,022 - INFO - Переход на страницу: /
2025-02-19 06:19:00,023 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("module 'flet' has no attribute 'NavigationDestination'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 6, in __init__
    super().__init__(page)
    ~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 13, in _build
    self._build_mobile_tablet()
    ~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 119, in _build_mobile_tablet
    ft.NavigationDestination(
    ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'flet' has no attribute 'NavigationDestination'. Did you mean: 'NavigationBarDestination'?
2025-02-19 06:19:01,261 - INFO - New FletApp: g6ki5dcq
2025-02-19 06:19:01,272 - INFO - New session created (2 total): _rxKnvDcUGYwGVYm6_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:19:01,272 - INFO - Start session: rxKnvDcUGYwGVYm6
2025-02-19 06:19:01,334 - INFO - New FletApp: y5GBcqpQ
2025-02-19 06:19:01,345 - INFO - New session created (3 total): _N1yTHyzSqTcyCZrx_6adfa3f7ac30bf9f7a7dc95e3e9e0f69152d3460
2025-02-19 06:19:01,345 - INFO - Start session: N1yTHyzSqTcyCZrx
2025-02-19 06:19:01,403 - INFO - Переход на страницу: /login
2025-02-19 06:19:01,406 - INFO - Переход на страницу: /
2025-02-19 06:19:01,409 - INFO - Переход на страницу: /login
2025-02-19 06:19:01,456 - INFO - Переход на страницу: /
2025-02-19 06:19:01,457 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("module 'flet' has no attribute 'NavigationDestination'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 6, in __init__
    super().__init__(page)
    ~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 13, in _build
    self._build_mobile_tablet()
    ~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 119, in _build_mobile_tablet
    ft.NavigationDestination(
    ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'flet' has no attribute 'NavigationDestination'. Did you mean: 'NavigationBarDestination'?
2025-02-19 06:19:04,664 - INFO - New FletApp: pMHsBOoH
2025-02-19 06:19:04,690 - INFO - New session created (4 total): _OGpM7U6ijTDv4HvD_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:19:04,690 - INFO - Start session: OGpM7U6ijTDv4HvD
2025-02-19 06:19:04,754 - INFO - Переход на страницу: /login
2025-02-19 06:19:04,817 - INFO - Переход на страницу: /
2025-02-19 06:19:04,818 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("module 'flet' has no attribute 'NavigationDestination'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 6, in __init__
    super().__init__(page)
    ~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 13, in _build
    self._build_mobile_tablet()
    ~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 119, in _build_mobile_tablet
    ft.NavigationDestination(
    ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'flet' has no attribute 'NavigationDestination'. Did you mean: 'NavigationBarDestination'?
2025-02-19 06:19:08,106 - INFO - Session disconnected: _N1yTHyzSqTcyCZrx_6adfa3f7ac30bf9f7a7dc95e3e9e0f69152d3460
2025-02-19 06:19:08,505 - INFO - New FletApp: cANLyxPF
2025-02-19 06:19:08,623 - INFO - Existing session requested: N1yTHyzSqTcyCZrx
2025-02-19 06:19:08,623 - INFO - Session reconnected: _N1yTHyzSqTcyCZrx_6adfa3f7ac30bf9f7a7dc95e3e9e0f69152d3460
2025-02-19 06:19:08,623 - INFO - Disposing FletApp: y5GBcqpQ
2025-02-19 06:19:12,633 - INFO - Session disconnected: _N1yTHyzSqTcyCZrx_6adfa3f7ac30bf9f7a7dc95e3e9e0f69152d3460
2025-02-19 06:19:14,427 - INFO - Session disconnected: _OGpM7U6ijTDv4HvD_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:19:15,657 - INFO - New FletApp: 1lss0Kve
2025-02-19 06:19:15,947 - INFO - Existing session requested: OGpM7U6ijTDv4HvD
2025-02-19 06:19:15,947 - INFO - Session reconnected: _OGpM7U6ijTDv4HvD_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:19:15,947 - INFO - Disposing FletApp: pMHsBOoH
2025-02-19 06:19:19,206 - INFO - Session disconnected: _rxKnvDcUGYwGVYm6_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:19:20,080 - INFO - New FletApp: G7F7fuen
2025-02-19 06:19:20,143 - INFO - Existing session requested: rxKnvDcUGYwGVYm6
2025-02-19 06:19:20,143 - INFO - Session reconnected: _rxKnvDcUGYwGVYm6_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:19:20,143 - INFO - Disposing FletApp: g6ki5dcq
2025-02-19 06:19:23,093 - INFO - Переход на страницу: /profile
2025-02-19 06:19:33,200 - INFO - Session disconnected: _rxKnvDcUGYwGVYm6_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:19:35,573 - INFO - New FletApp: 4QBXBkXm
2025-02-19 06:19:35,700 - INFO - New session created (5 total): _sQHmW7Fq6y17IAg2_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:19:35,701 - INFO - Start session: sQHmW7Fq6y17IAg2
2025-02-19 06:19:35,768 - INFO - Переход на страницу: /login
2025-02-19 06:19:35,817 - INFO - Переход на страницу: /
2025-02-19 06:19:35,818 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("module 'flet' has no attribute 'NavigationDestination'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 6, in __init__
    super().__init__(page)
    ~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 13, in _build
    self._build_mobile_tablet()
    ~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 119, in _build_mobile_tablet
    ft.NavigationDestination(
    ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'flet' has no attribute 'NavigationDestination'. Did you mean: 'NavigationBarDestination'?
2025-02-19 06:19:54,845 - INFO - Session disconnected: _sQHmW7Fq6y17IAg2_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:19:57,804 - INFO - New FletApp: LMzbpBbf
2025-02-19 06:19:58,024 - INFO - Existing session requested: sQHmW7Fq6y17IAg2
2025-02-19 06:19:58,024 - INFO - Session reconnected: _sQHmW7Fq6y17IAg2_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:19:58,024 - INFO - Disposing FletApp: 4QBXBkXm
2025-02-19 06:20:01,217 - INFO - Session disconnected: _sQHmW7Fq6y17IAg2_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:20:03,488 - INFO - New FletApp: FkWQqrzx
2025-02-19 06:20:03,803 - INFO - Existing session requested: sQHmW7Fq6y17IAg2
2025-02-19 06:20:03,804 - INFO - Session reconnected: _sQHmW7Fq6y17IAg2_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:20:03,804 - INFO - Disposing FletApp: LMzbpBbf
2025-02-19 06:20:09,089 - INFO - Session disconnected: _sQHmW7Fq6y17IAg2_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:20:10,734 - INFO - New FletApp: dw8q6vPZ
2025-02-19 06:20:12,342 - INFO - New session created (6 total): _GKieLb1JN6VIlBkj_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:20:12,343 - INFO - Start session: GKieLb1JN6VIlBkj
2025-02-19 06:20:12,365 - INFO - Переход на страницу: /login
2025-02-19 06:20:12,381 - INFO - Переход на страницу: /
2025-02-19 06:20:12,386 - INFO - Переход на страницу: /login
2025-02-19 06:20:12,438 - INFO - Переход на страницу: /
2025-02-19 06:20:14,264 - INFO - Session disconnected: _GKieLb1JN6VIlBkj_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:20:15,717 - INFO - New FletApp: l6r6xuhV
2025-02-19 06:20:15,832 - INFO - Existing session requested: GKieLb1JN6VIlBkj
2025-02-19 06:20:15,832 - INFO - Session reconnected: _GKieLb1JN6VIlBkj_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:20:15,832 - INFO - Disposing FletApp: dw8q6vPZ
2025-02-19 06:20:19,062 - INFO - Session disconnected: _GKieLb1JN6VIlBkj_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:20:19,062 - INFO - Session disconnected: _vv4dtiqdme4k4kH4_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:20:19,062 - INFO - Session disconnected: _OGpM7U6ijTDv4HvD_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:20:19,169 - INFO - Shutting down Flet App Manager
2025-02-19 06:20:19,170 - INFO - Deleting temp dir: C:\Temp\tmpq1hdgkuk
2025-02-19 06:20:19,171 - INFO - Flet app has started...
2025-02-19 06:20:19,171 - INFO - Closing Flet web server...
2025-02-19 06:20:28,100 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:20:28,545 - INFO - Starting Flet web server on port 8550...
2025-02-19 06:20:28,546 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 06:20:28,546 - INFO - Use color emoji: False
2025-02-19 06:20:28,546 - INFO - Route URL strategy configured: path
2025-02-19 06:20:28,579 - INFO - Starting up Flet App Manager
2025-02-19 06:20:28,580 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 06:20:28,930 - INFO - New FletApp: 80NDVXnc
2025-02-19 06:20:29,025 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmpf7aql1l9
2025-02-19 06:20:29,025 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 06:20:29,026 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:20:29,180 - INFO - New FletApp: 1C3LJzd3
2025-02-19 06:20:29,264 - INFO - New session created (1 total): _bWeVoaSt0KMvEt3u_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:20:29,264 - INFO - Start session: bWeVoaSt0KMvEt3u
2025-02-19 06:20:29,278 - INFO - Переход на страницу: /login
2025-02-19 06:20:29,332 - INFO - Переход на страницу: /
2025-02-19 06:20:29,333 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("module 'flet' has no attribute 'NavigationDestination'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 6, in __init__
    super().__init__(page)
    ~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 13, in _build
    self._build_mobile_tablet()
    ~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 119, in _build_mobile_tablet
    ft.NavigationDestination(
    ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'flet' has no attribute 'NavigationDestination'. Did you mean: 'NavigationBarDestination'?
2025-02-19 06:20:32,575 - INFO - New FletApp: 99QVOJHx
2025-02-19 06:20:32,697 - INFO - New session created (2 total): _ZPCl3ondn0RQoUOu_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:20:32,697 - INFO - Start session: ZPCl3ondn0RQoUOu
2025-02-19 06:20:32,712 - INFO - Переход на страницу: /login
2025-02-19 06:20:32,715 - INFO - Переход на страницу: /
2025-02-19 06:20:33,241 - INFO - Session disconnected: _bWeVoaSt0KMvEt3u_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:20:34,703 - INFO - New FletApp: gwm8c3MQ
2025-02-19 06:20:34,716 - INFO - New session created (3 total): _A2imT3AhSihNxtTJ_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:20:34,717 - INFO - Start session: A2imT3AhSihNxtTJ
2025-02-19 06:20:34,780 - INFO - Переход на страницу: /login
2025-02-19 06:20:34,828 - INFO - Переход на страницу: /
2025-02-19 06:20:34,829 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("module 'flet' has no attribute 'NavigationDestination'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 6, in __init__
    super().__init__(page)
    ~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 13, in _build
    self._build_mobile_tablet()
    ~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 119, in _build_mobile_tablet
    ft.NavigationDestination(
    ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'flet' has no attribute 'NavigationDestination'. Did you mean: 'NavigationBarDestination'?
2025-02-19 06:20:37,603 - INFO - Session disconnected: _ZPCl3ondn0RQoUOu_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:20:39,326 - INFO - New FletApp: PSSGBn0v
2025-02-19 06:20:39,351 - INFO - Existing session requested: ZPCl3ondn0RQoUOu
2025-02-19 06:20:39,351 - INFO - Session reconnected: _ZPCl3ondn0RQoUOu_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:20:39,351 - INFO - Disposing FletApp: 99QVOJHx
2025-02-19 06:21:02,164 - INFO - Переход на страницу: /profile
2025-02-19 06:21:12,865 - INFO - Session disconnected: _ZPCl3ondn0RQoUOu_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:23:25,387 - INFO - Session disconnected: _A2imT3AhSihNxtTJ_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:23:25,495 - INFO - Shutting down Flet App Manager
2025-02-19 06:23:25,495 - INFO - Deleting temp dir: C:\Temp\tmpf7aql1l9
2025-02-19 06:23:25,496 - INFO - Flet app has started...
2025-02-19 06:23:25,496 - INFO - Closing Flet web server...
2025-02-19 06:23:32,796 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:23:32,798 - INFO - Starting up TCP server on localhost:54044
2025-02-19 06:23:32,805 - INFO - Flet app has started...
2025-02-19 06:23:33,098 - INFO - App session started
2025-02-19 06:23:33,102 - INFO - Переход на страницу: /login
2025-02-19 06:23:33,104 - INFO - Переход на страницу: /
2025-02-19 06:23:35,498 - INFO - Переход на страницу: /settings
2025-02-19 06:23:36,959 - INFO - Переход на страницу: /
2025-02-19 06:23:48,699 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:23:48,701 - INFO - Starting up TCP server on localhost:54053
2025-02-19 06:23:48,707 - INFO - Flet app has started...
2025-02-19 06:23:48,710 - INFO - App URL: tcp://localhost:54053
2025-02-19 06:23:48,710 - INFO - Starting Flet View app...
2025-02-19 06:23:48,711 - INFO - Looking for Flet executable at: C:\Program Files\Python313\Lib\site-packages\flet_desktop\app\flet\flet.exe
2025-02-19 06:23:48,711 - INFO - Flet View found in: C:\Program Files\Python313\Lib\site-packages\flet_desktop\app\flet\flet.exe
2025-02-19 06:23:48,978 - INFO - App session started
2025-02-19 06:23:48,981 - INFO - Переход на страницу: /login
2025-02-19 06:23:48,984 - INFO - Переход на страницу: /
2025-02-19 06:24:05,421 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:24:05,800 - INFO - Starting Flet web server on port 54058...
2025-02-19 06:24:05,801 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 06:24:05,801 - INFO - Use color emoji: False
2025-02-19 06:24:05,801 - INFO - Route URL strategy configured: path
2025-02-19 06:24:05,828 - INFO - Starting up Flet App Manager
2025-02-19 06:24:06,100 - INFO - New FletApp: xrhAduyV
2025-02-19 06:24:06,103 - INFO - New session created (1 total): _7hijVeOz6ft0fen6_1c937ecbeba32347b08c451ce625e216d4357604
2025-02-19 06:24:06,103 - INFO - Start session: 7hijVeOz6ft0fen6
2025-02-19 06:24:06,107 - INFO - Переход на страницу: /login
2025-02-19 06:24:06,110 - INFO - Переход на страницу: /
2025-02-19 06:24:10,233 - INFO - Session disconnected: _7hijVeOz6ft0fen6_1c937ecbeba32347b08c451ce625e216d4357604
2025-02-19 06:24:21,215 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:24:21,602 - INFO - Starting Flet web server on port 54065...
2025-02-19 06:24:21,603 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 06:24:21,603 - INFO - Use color emoji: False
2025-02-19 06:24:21,603 - INFO - Route URL strategy configured: path
2025-02-19 06:24:21,634 - INFO - Starting up Flet App Manager
2025-02-19 06:24:22,008 - INFO - New FletApp: 5kpnR7vt
2025-02-19 06:24:22,011 - INFO - New session created (1 total): _QUHdKNHEkmGVL6VO_1c937ecbeba32347b08c451ce625e216d4357604
2025-02-19 06:24:22,011 - INFO - Start session: QUHdKNHEkmGVL6VO
2025-02-19 06:24:22,016 - INFO - Переход на страницу: /login
2025-02-19 06:24:22,018 - INFO - Переход на страницу: /
2025-02-19 06:24:26,180 - INFO - Session disconnected: _QUHdKNHEkmGVL6VO_1c937ecbeba32347b08c451ce625e216d4357604
2025-02-19 06:24:37,495 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:24:37,875 - INFO - Starting Flet web server on port 54073...
2025-02-19 06:24:37,876 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 06:24:37,876 - INFO - Use color emoji: False
2025-02-19 06:24:37,877 - INFO - Route URL strategy configured: path
2025-02-19 06:24:37,904 - INFO - Starting up Flet App Manager
2025-02-19 06:24:38,187 - INFO - New FletApp: awA7WklI
2025-02-19 06:24:38,190 - INFO - New session created (1 total): _JF4C82tQ6yrZFl4E_1c937ecbeba32347b08c451ce625e216d4357604
2025-02-19 06:24:38,190 - INFO - Start session: JF4C82tQ6yrZFl4E
2025-02-19 06:24:38,195 - INFO - Переход на страницу: /login
2025-02-19 06:24:38,197 - INFO - Переход на страницу: /
2025-02-19 06:25:21,793 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmpd3wtgn_h
2025-02-19 06:25:21,793 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 06:25:21,793 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:25:23,066 - INFO - New FletApp: kt5zmt8d
2025-02-19 06:25:23,307 - INFO - New session created (2 total): _0oAp2xyG0ra7D7Ac_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:25:23,307 - INFO - Start session: 0oAp2xyG0ra7D7Ac
2025-02-19 06:25:23,350 - INFO - Переход на страницу: /login
2025-02-19 06:25:29,148 - INFO - Переход на страницу: /
2025-02-19 06:25:32,319 - INFO - Переход на страницу: /settings
2025-02-19 06:25:34,534 - INFO - Переход на страницу: /
2025-02-19 06:25:41,553 - INFO - Переход на страницу: /settings
2025-02-19 06:25:47,433 - INFO - Переход на страницу: /
2025-02-19 06:25:48,850 - INFO - Переход на страницу: /settings
2025-02-19 06:25:49,730 - INFO - Переход на страницу: /login
2025-02-19 06:25:54,439 - INFO - Переход на страницу: /
2025-02-19 06:26:10,439 - INFO - Session disconnected: _0oAp2xyG0ra7D7Ac_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:26:11,298 - INFO - New FletApp: bM9MQn2O
2025-02-19 06:26:11,455 - INFO - Existing session requested: 0oAp2xyG0ra7D7Ac
2025-02-19 06:26:11,456 - INFO - Session reconnected: _0oAp2xyG0ra7D7Ac_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:26:11,456 - INFO - Disposing FletApp: kt5zmt8d
2025-02-19 06:26:13,077 - INFO - Переход на страницу: /settings
2025-02-19 06:26:14,485 - INFO - Переход на страницу: /
2025-02-19 06:26:56,431 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:26:56,826 - INFO - Starting Flet web server on port 54073...
2025-02-19 06:26:56,827 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 06:26:56,827 - INFO - Use color emoji: False
2025-02-19 06:26:56,827 - INFO - Route URL strategy configured: path
2025-02-19 06:26:56,857 - INFO - Starting up Flet App Manager
2025-02-19 06:26:57,112 - INFO - New FletApp: 2Di59C2q
2025-02-19 06:26:57,115 - INFO - New session created (1 total): _lEpzJGdwDVQYWAyI_1c937ecbeba32347b08c451ce625e216d4357604
2025-02-19 06:26:57,115 - INFO - Start session: lEpzJGdwDVQYWAyI
2025-02-19 06:26:57,118 - INFO - Переход на страницу: /login
2025-02-19 06:26:57,125 - INFO - Переход на страницу: /
2025-02-19 06:26:58,785 - INFO - New FletApp: 7ivFZ6nF
2025-02-19 06:26:58,801 - INFO - New session created (2 total): _Y3pl7V7W3isPADyd_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:26:58,801 - INFO - Start session: Y3pl7V7W3isPADyd
2025-02-19 06:26:58,868 - INFO - Переход на страницу: /login
2025-02-19 06:26:58,914 - INFO - Переход на страницу: /
2025-02-19 06:26:58,915 - ERROR - Future exception was never retrieved
future: <Future finished exception=TypeError("BorderRadius.__init__() missing 3 required positional arguments: 'top_right', 'bottom_left', and 'bottom_right'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 39, in _build
    border_radius=ft.BorderRadius(20),
                  ~~~~~~~~~~~~~~~^^^^
TypeError: BorderRadius.__init__() missing 3 required positional arguments: 'top_right', 'bottom_left', and 'bottom_right'
2025-02-19 06:27:02,053 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmptpl7esx1
2025-02-19 06:27:02,054 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 06:27:02,054 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:27:02,107 - INFO - Session disconnected: _Y3pl7V7W3isPADyd_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:27:03,622 - INFO - New FletApp: KYIjuEwr
2025-02-19 06:27:03,845 - INFO - Existing session requested: Y3pl7V7W3isPADyd
2025-02-19 06:27:03,845 - INFO - Session reconnected: _Y3pl7V7W3isPADyd_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:27:03,846 - INFO - Disposing FletApp: 7ivFZ6nF
2025-02-19 06:27:03,879 - INFO - Переход на страницу: /login
2025-02-19 06:27:03,882 - INFO - Переход на страницу: /
2025-02-19 06:27:03,882 - ERROR - Future exception was never retrieved
future: <Future finished exception=TypeError("BorderRadius.__init__() missing 3 required positional arguments: 'top_right', 'bottom_left', and 'bottom_right'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 39, in _build
    border_radius=ft.BorderRadius(20),
                  ~~~~~~~~~~~~~~~^^^^
TypeError: BorderRadius.__init__() missing 3 required positional arguments: 'top_right', 'bottom_left', and 'bottom_right'
2025-02-19 06:27:05,268 - INFO - Переход на страницу: /login
2025-02-19 06:27:05,287 - INFO - Переход на страницу: /
2025-02-19 06:27:05,288 - ERROR - Future exception was never retrieved
future: <Future finished exception=TypeError("BorderRadius.__init__() missing 3 required positional arguments: 'top_right', 'bottom_left', and 'bottom_right'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 39, in _build
    border_radius=ft.BorderRadius(20),
                  ~~~~~~~~~~~~~~~^^^^
TypeError: BorderRadius.__init__() missing 3 required positional arguments: 'top_right', 'bottom_left', and 'bottom_right'
2025-02-19 06:27:06,448 - INFO - Session disconnected: _Y3pl7V7W3isPADyd_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:27:08,520 - INFO - New FletApp: oOU57jjy
2025-02-19 06:27:08,738 - INFO - New session created (3 total): _xuMSJOlVIckw528q_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:27:08,738 - INFO - Start session: xuMSJOlVIckw528q
2025-02-19 06:27:08,782 - INFO - Переход на страницу: /login
2025-02-19 06:27:08,856 - INFO - Переход на страницу: /
2025-02-19 06:27:08,857 - ERROR - Future exception was never retrieved
future: <Future finished exception=TypeError("BorderRadius.__init__() missing 3 required positional arguments: 'top_right', 'bottom_left', and 'bottom_right'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 39, in _build
    border_radius=ft.BorderRadius(20),
                  ~~~~~~~~~~~~~~~^^^^
TypeError: BorderRadius.__init__() missing 3 required positional arguments: 'top_right', 'bottom_left', and 'bottom_right'
2025-02-19 06:28:21,870 - INFO - Session disconnected: _xuMSJOlVIckw528q_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:28:22,655 - INFO - New FletApp: hoz0ze4f
2025-02-19 06:28:22,835 - INFO - Existing session requested: xuMSJOlVIckw528q
2025-02-19 06:28:22,835 - INFO - Session reconnected: _xuMSJOlVIckw528q_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:28:22,835 - INFO - Disposing FletApp: oOU57jjy
2025-02-19 06:28:27,466 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:28:27,803 - INFO - Starting Flet web server on port 54073...
2025-02-19 06:28:27,803 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 06:28:27,804 - INFO - Use color emoji: False
2025-02-19 06:28:27,804 - INFO - Route URL strategy configured: path
2025-02-19 06:28:27,828 - INFO - Starting up Flet App Manager
2025-02-19 06:28:28,190 - INFO - New FletApp: uf7oUJqj
2025-02-19 06:28:28,191 - INFO - New session created (1 total): _WxTZlVrMfCkXIAJg_1c937ecbeba32347b08c451ce625e216d4357604
2025-02-19 06:28:28,192 - INFO - Start session: WxTZlVrMfCkXIAJg
2025-02-19 06:28:28,194 - INFO - Переход на страницу: /login
2025-02-19 06:28:28,195 - INFO - Переход на страницу: /
2025-02-19 06:28:29,823 - INFO - New FletApp: Uq3nSbbR
2025-02-19 06:28:29,837 - INFO - New session created (2 total): _6wefrMYYrsYUyAo4_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:28:29,837 - INFO - Start session: 6wefrMYYrsYUyAo4
2025-02-19 06:28:29,895 - INFO - Переход на страницу: /login
2025-02-19 06:28:29,945 - INFO - Переход на страницу: /
2025-02-19 06:28:29,945 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("module 'flet' has no attribute 'Expanded'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 27, in _build
    ft.Expanded(
    ^^^^^^^^^^^
AttributeError: module 'flet' has no attribute 'Expanded'
2025-02-19 06:30:38,585 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:30:38,960 - INFO - Starting Flet web server on port 54189...
2025-02-19 06:30:38,962 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 06:30:38,962 - INFO - Use color emoji: False
2025-02-19 06:30:38,962 - INFO - Route URL strategy configured: path
2025-02-19 06:30:38,988 - INFO - Starting up Flet App Manager
2025-02-19 06:30:39,256 - INFO - New FletApp: YsxTtaKE
2025-02-19 06:30:39,258 - INFO - New session created (1 total): _BVODTz6H4GRKN3XX_1c937ecbeba32347b08c451ce625e216d4357604
2025-02-19 06:30:39,258 - INFO - Start session: BVODTz6H4GRKN3XX
2025-02-19 06:30:39,263 - INFO - Переход на страницу: /login
2025-02-19 06:30:39,265 - INFO - Переход на страницу: /
2025-02-19 06:30:53,663 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmp6up09hqj
2025-02-19 06:30:53,663 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 06:30:53,664 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:30:54,967 - INFO - New FletApp: 9YizT1PR
2025-02-19 06:30:55,178 - INFO - New session created (2 total): _YgPC0sxW0clu1G0t_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:30:55,178 - INFO - Start session: YgPC0sxW0clu1G0t
2025-02-19 06:30:55,222 - INFO - Переход на страницу: /login
2025-02-19 06:31:01,078 - INFO - Переход на страницу: /
2025-02-19 06:31:01,080 - ERROR - Future exception was never retrieved
future: <Future finished exception=TypeError("BorderRadius.__init__() missing 3 required positional arguments: 'top_right', 'bottom_left', and 'bottom_right'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 38, in _build
    border_radius=ft.BorderRadius(20),
                  ~~~~~~~~~~~~~~~^^^^
TypeError: BorderRadius.__init__() missing 3 required positional arguments: 'top_right', 'bottom_left', and 'bottom_right'
2025-02-19 06:31:02,823 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("'NoneType' object has no attribute 'client_storage'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\login.py", line 31, in _handle_login
    self.page.client_storage.set("logged_in", True)
    ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'client_storage'
2025-02-19 06:31:12,730 - INFO - Session disconnected: _YgPC0sxW0clu1G0t_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:31:14,508 - INFO - New FletApp: j25aPl5h
2025-02-19 06:31:14,623 - INFO - Existing session requested: YgPC0sxW0clu1G0t
2025-02-19 06:31:14,623 - INFO - Session reconnected: _YgPC0sxW0clu1G0t_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:31:14,623 - INFO - Disposing FletApp: 9YizT1PR
2025-02-19 06:31:19,369 - INFO - Session disconnected: _YgPC0sxW0clu1G0t_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:31:19,941 - INFO - New FletApp: YsU74QiZ
2025-02-19 06:31:20,078 - INFO - Existing session requested: YgPC0sxW0clu1G0t
2025-02-19 06:31:20,078 - INFO - Session reconnected: _YgPC0sxW0clu1G0t_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:31:20,078 - INFO - Disposing FletApp: j25aPl5h
2025-02-19 06:31:33,067 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:31:33,410 - INFO - Starting Flet web server on port 54189...
2025-02-19 06:31:33,411 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 06:31:33,411 - INFO - Use color emoji: False
2025-02-19 06:31:33,411 - INFO - Route URL strategy configured: path
2025-02-19 06:31:33,435 - INFO - Starting up Flet App Manager
2025-02-19 06:31:33,802 - INFO - New FletApp: LCFNDGBQ
2025-02-19 06:31:33,805 - INFO - New session created (1 total): _Bpeww1q1l3HxSpOI_1c937ecbeba32347b08c451ce625e216d4357604
2025-02-19 06:31:33,805 - INFO - Start session: Bpeww1q1l3HxSpOI
2025-02-19 06:31:33,808 - INFO - Переход на страницу: /login
2025-02-19 06:31:33,809 - INFO - Переход на страницу: /
2025-02-19 06:31:33,812 - INFO - Переход на страницу: /login
2025-02-19 06:31:33,813 - INFO - Переход на страницу: /
2025-02-19 06:31:35,487 - INFO - New FletApp: VXX26VsR
2025-02-19 06:31:35,498 - INFO - New session created (2 total): _reMQditzDKVTHHel_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:31:35,498 - INFO - Start session: reMQditzDKVTHHel
2025-02-19 06:31:35,574 - INFO - Переход на страницу: /login
2025-02-19 06:31:35,649 - INFO - Переход на страницу: /
2025-02-19 06:32:35,067 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:32:35,407 - INFO - Starting Flet web server on port 54189...
2025-02-19 06:32:35,408 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 06:32:35,408 - INFO - Use color emoji: False
2025-02-19 06:32:35,408 - INFO - Route URL strategy configured: path
2025-02-19 06:32:35,432 - INFO - Starting up Flet App Manager
2025-02-19 06:32:35,492 - INFO - New FletApp: 3JxbAgLU
2025-02-19 06:32:35,499 - INFO - New session created (1 total): _Q9ka563sskL1lK7W_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:32:35,500 - INFO - Start session: Q9ka563sskL1lK7W
2025-02-19 06:32:35,557 - INFO - Переход на страницу: /login
2025-02-19 06:32:35,615 - INFO - Переход на страницу: /
2025-02-19 06:32:35,615 - ERROR - Future exception was never retrieved
future: <Future finished exception=TypeError("Column.__init__() got an unexpected keyword argument 'bgcolor'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 26, in _build
    ft.Column([
    ~~~~~~~~~^^
        ft.Text("Welcome to the Mobile Home Page!", size=24, weight=ft.FontWeight.BOLD),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        ft.Text(f"You're using a {device_type}.", size=18),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        ft.ElevatedButton("Go to Mobile Settings", on_click=lambda _: self.page.go("/settings"))
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ], spacing=10, alignment=ft.MainAxisAlignment.START, bgcolor="black")
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Column.__init__() got an unexpected keyword argument 'bgcolor'
2025-02-19 06:32:35,846 - INFO - New FletApp: xrU1Fvyj
2025-02-19 06:32:35,849 - INFO - New session created (2 total): _njL2bjRxrUFoOzgr_1c937ecbeba32347b08c451ce625e216d4357604
2025-02-19 06:32:35,849 - INFO - Start session: njL2bjRxrUFoOzgr
2025-02-19 06:32:35,850 - INFO - Переход на страницу: /login
2025-02-19 06:32:35,852 - INFO - Переход на страницу: /
2025-02-19 06:32:37,802 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmp3z473n_f
2025-02-19 06:32:37,802 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 06:32:37,802 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:32:37,853 - INFO - Session disconnected: _Q9ka563sskL1lK7W_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:32:38,819 - INFO - New FletApp: PR30BrKf
2025-02-19 06:32:39,005 - INFO - Existing session requested: Q9ka563sskL1lK7W
2025-02-19 06:32:39,006 - INFO - Session reconnected: _Q9ka563sskL1lK7W_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:32:39,006 - INFO - Disposing FletApp: 3JxbAgLU
2025-02-19 06:32:57,604 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:32:57,958 - INFO - Starting Flet web server on port 54189...
2025-02-19 06:32:57,959 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 06:32:57,959 - INFO - Use color emoji: False
2025-02-19 06:32:57,959 - INFO - Route URL strategy configured: path
2025-02-19 06:32:57,982 - INFO - Starting up Flet App Manager
2025-02-19 06:32:58,095 - INFO - New FletApp: CgzRcNq5
2025-02-19 06:32:58,108 - INFO - New session created (1 total): _GuzJIx75MbFK8Wny_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:32:58,108 - INFO - Start session: GuzJIx75MbFK8Wny
2025-02-19 06:32:58,172 - INFO - Переход на страницу: /login
2025-02-19 06:32:58,220 - INFO - Переход на страницу: /
2025-02-19 06:32:58,357 - INFO - New FletApp: 4ZaOUJxV
2025-02-19 06:32:58,360 - INFO - New session created (2 total): _Sp6PoSQhflmYmiOz_1c937ecbeba32347b08c451ce625e216d4357604
2025-02-19 06:32:58,360 - INFO - Start session: Sp6PoSQhflmYmiOz
2025-02-19 06:32:58,361 - INFO - Переход на страницу: /login
2025-02-19 06:32:58,363 - INFO - Переход на страницу: /
2025-02-19 06:33:31,299 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:33:31,643 - INFO - Starting Flet web server on port 54189...
2025-02-19 06:33:31,643 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 06:33:31,643 - INFO - Use color emoji: False
2025-02-19 06:33:31,643 - INFO - Route URL strategy configured: path
2025-02-19 06:33:31,666 - INFO - Starting up Flet App Manager
2025-02-19 06:33:31,719 - INFO - New FletApp: ec3W0zI9
2025-02-19 06:33:31,732 - INFO - New session created (1 total): _Yn8nsX3zGIPosIsO_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:33:31,733 - INFO - Start session: Yn8nsX3zGIPosIsO
2025-02-19 06:33:31,794 - INFO - Переход на страницу: /login
2025-02-19 06:33:31,846 - INFO - Переход на страницу: /
2025-02-19 06:33:31,847 - ERROR - Future exception was never retrieved
future: <Future finished exception=TypeError("Column.__init__() got an unexpected keyword argument 'bgcolor'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 26, in _build
    ft.Column([
    ~~~~~~~~~^^
        ft.Text("Welcome to the Mobile Home Page!", size=24, weight=ft.FontWeight.BOLD),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        ft.Text(f"You're using a {device_type}.", size=18),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        ft.ElevatedButton("Go to Mobile Settings", on_click=lambda _: self.page.go("/settings"))
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ], spacing=10, bgcolor=ft.Colors.GREY_200, alignment=ft.MainAxisAlignment.START)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Column.__init__() got an unexpected keyword argument 'bgcolor'
2025-02-19 06:33:32,054 - INFO - New FletApp: HrRsS6LV
2025-02-19 06:33:32,055 - INFO - New session created (2 total): _52b5qGvLF4NySlji_1c937ecbeba32347b08c451ce625e216d4357604
2025-02-19 06:33:32,055 - INFO - Start session: 52b5qGvLF4NySlji
2025-02-19 06:33:32,057 - INFO - Переход на страницу: /login
2025-02-19 06:33:32,058 - INFO - Переход на страницу: /
2025-02-19 06:33:33,599 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmps9ms3uro
2025-02-19 06:33:33,599 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 06:33:33,599 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:33:33,652 - INFO - Session disconnected: _Yn8nsX3zGIPosIsO_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:33:34,710 - INFO - New FletApp: tncvyUaj
2025-02-19 06:33:34,983 - INFO - Existing session requested: Yn8nsX3zGIPosIsO
2025-02-19 06:33:34,983 - INFO - Session reconnected: _Yn8nsX3zGIPosIsO_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:33:34,983 - INFO - Disposing FletApp: ec3W0zI9
2025-02-19 06:34:04,924 - INFO - Session disconnected: _52b5qGvLF4NySlji_1c937ecbeba32347b08c451ce625e216d4357604
2025-02-19 06:34:04,939 - INFO - Session disconnected: _Yn8nsX3zGIPosIsO_2997bd35ae41b72d0e37e46e9f311cdd7e8a22d8
2025-02-19 06:34:09,336 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:34:13,717 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:34:14,095 - INFO - Starting Flet web server on port 8550...
2025-02-19 06:34:14,096 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 06:34:14,096 - INFO - Use color emoji: False
2025-02-19 06:34:14,096 - INFO - Route URL strategy configured: path
2025-02-19 06:34:14,121 - INFO - Starting up Flet App Manager
2025-02-19 06:34:14,121 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 06:34:14,502 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmpixv4mala
2025-02-19 06:34:14,502 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 06:34:14,502 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:34:15,684 - INFO - New FletApp: oCtpzg9G
2025-02-19 06:34:15,701 - INFO - New session created (1 total): _vG1VzFs6Yqj1WHtA_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:34:15,701 - INFO - Start session: vG1VzFs6Yqj1WHtA
2025-02-19 06:34:15,818 - INFO - Переход на страницу: /login
2025-02-19 06:34:15,825 - INFO - Переход на страницу: /
2025-02-19 06:34:26,063 - INFO - Session disconnected: _vG1VzFs6Yqj1WHtA_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:34:28,250 - INFO - New FletApp: az80SSMO
2025-02-19 06:34:28,413 - INFO - New session created (2 total): _UTHV2zZjkWCxyTkk_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:34:28,414 - INFO - Start session: UTHV2zZjkWCxyTkk
2025-02-19 06:34:28,459 - INFO - Переход на страницу: /login
2025-02-19 06:34:28,509 - INFO - Переход на страницу: /
2025-02-19 06:34:28,510 - ERROR - Future exception was never retrieved
future: <Future finished exception=TypeError("Column.__init__() got an unexpected keyword argument 'bgcolor'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 26, in _build
    ft.Column([
    ~~~~~~~~~^^
        ft.Text("Welcome to the Mobile Home Page!", size=24, weight=ft.FontWeight.BOLD),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        ft.Text(f"You're using a {device_type}.", size=18),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        ft.ElevatedButton("Go to Mobile Settings", on_click=lambda _: self.page.go("/settings"))
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ], spacing=10, bgcolor=ft.Colors.GREY_200, alignment=ft.MainAxisAlignment.START)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Column.__init__() got an unexpected keyword argument 'bgcolor'
2025-02-19 06:34:34,754 - INFO - Session disconnected: _UTHV2zZjkWCxyTkk_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:34:36,293 - INFO - New FletApp: 8lfE3qDb
2025-02-19 06:34:36,478 - INFO - Existing session requested: UTHV2zZjkWCxyTkk
2025-02-19 06:34:36,479 - INFO - Session reconnected: _UTHV2zZjkWCxyTkk_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:34:36,479 - INFO - Disposing FletApp: az80SSMO
2025-02-19 06:35:35,757 - INFO - Session disconnected: _UTHV2zZjkWCxyTkk_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:35:35,864 - INFO - Shutting down Flet App Manager
2025-02-19 06:35:35,864 - INFO - Deleting temp dir: C:\Temp\tmpixv4mala
2025-02-19 06:35:35,865 - INFO - Flet app has started...
2025-02-19 06:35:35,866 - INFO - Closing Flet web server...
2025-02-19 06:35:41,134 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:35:41,562 - INFO - Starting Flet web server on port 8550...
2025-02-19 06:35:41,563 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 06:35:41,563 - INFO - Use color emoji: False
2025-02-19 06:35:41,563 - INFO - Route URL strategy configured: path
2025-02-19 06:35:41,585 - INFO - Starting up Flet App Manager
2025-02-19 06:35:41,585 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 06:35:41,789 - INFO - New FletApp: cq52wZBM
2025-02-19 06:35:42,045 - INFO - New session created (1 total): _br8zYoj3krTeXZuR_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:35:42,046 - INFO - Start session: br8zYoj3krTeXZuR
2025-02-19 06:35:42,053 - INFO - Переход на страницу: /login
2025-02-19 06:35:42,273 - INFO - Переход на страницу: /
2025-02-19 06:35:42,274 - ERROR - Future exception was never retrieved
future: <Future finished exception=TypeError("Column.__init__() got an unexpected keyword argument 'bgcolor'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 26, in _build
    ft.Column([
    ~~~~~~~~~^^
        ft.Text("Welcome to the Mobile Home Page!", size=24, weight=ft.FontWeight.BOLD),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        ft.Text(f"You're using a {device_type}.", size=18),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        ft.ElevatedButton("Go to Mobile Settings", on_click=lambda _: self.page.go("/settings"))
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ], expand=True, spacing=10, bgcolor=ft.Colors.GREY_200, alignment=ft.MainAxisAlignment.START)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Column.__init__() got an unexpected keyword argument 'bgcolor'
2025-02-19 06:35:42,340 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmpwiq718fp
2025-02-19 06:35:42,340 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 06:35:42,340 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:35:44,674 - INFO - New FletApp: THaQORQQ
2025-02-19 06:35:44,681 - INFO - New session created (2 total): _dbhFbJxU6PtMneeQ_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:35:44,681 - INFO - Start session: dbhFbJxU6PtMneeQ
2025-02-19 06:35:44,686 - INFO - Переход на страницу: /login
2025-02-19 06:35:44,689 - INFO - Переход на страницу: /
2025-02-19 06:35:44,966 - INFO - Session disconnected: _br8zYoj3krTeXZuR_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:35:46,560 - INFO - New FletApp: mkQkPIdZ
2025-02-19 06:35:46,755 - INFO - Existing session requested: br8zYoj3krTeXZuR
2025-02-19 06:35:46,755 - INFO - Session reconnected: _br8zYoj3krTeXZuR_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:35:46,755 - INFO - Disposing FletApp: cq52wZBM
2025-02-19 06:35:56,903 - INFO - Session disconnected: _br8zYoj3krTeXZuR_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:35:58,616 - INFO - New FletApp: Jp1RMSVD
2025-02-19 06:35:58,813 - INFO - Existing session requested: br8zYoj3krTeXZuR
2025-02-19 06:35:58,814 - INFO - Session reconnected: _br8zYoj3krTeXZuR_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:35:58,814 - INFO - Disposing FletApp: mkQkPIdZ
2025-02-19 06:36:04,939 - INFO - Session disconnected: _br8zYoj3krTeXZuR_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:36:07,156 - INFO - New FletApp: gQIrWSOU
2025-02-19 06:36:07,383 - INFO - Existing session requested: br8zYoj3krTeXZuR
2025-02-19 06:36:07,384 - INFO - Session reconnected: _br8zYoj3krTeXZuR_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:36:07,384 - INFO - Disposing FletApp: Jp1RMSVD
2025-02-19 06:36:11,737 - INFO - Session disconnected: _br8zYoj3krTeXZuR_3e70c86fb633be54f66517e3bec4726410890ae7
2025-02-19 06:36:13,418 - INFO - New FletApp: bx55Owa7
2025-02-19 06:36:13,584 - INFO - New session created (3 total): _BvaclCEIDxJSPvEU_3b72f58663583f7baae0cdd1dc76682f7cc73967
2025-02-19 06:36:13,585 - INFO - Start session: BvaclCEIDxJSPvEU
2025-02-19 06:36:13,644 - INFO - Переход на страницу: /login
2025-02-19 06:36:13,728 - INFO - Переход на страницу: /
2025-02-19 06:36:13,729 - INFO - Переход на страницу: /login
2025-02-19 06:36:13,845 - INFO - Переход на страницу: /
2025-02-19 06:36:18,752 - INFO - Session disconnected: _BvaclCEIDxJSPvEU_3b72f58663583f7baae0cdd1dc76682f7cc73967
2025-02-19 06:36:20,665 - INFO - New FletApp: W3J4pKce
2025-02-19 06:36:20,987 - INFO - New session created (4 total): _N0e3xPCE4LxF1NtH_7e65f042b7c94ff8759fa7834cb934484151af3d
2025-02-19 06:36:20,987 - INFO - Start session: N0e3xPCE4LxF1NtH
2025-02-19 06:36:21,036 - INFO - Переход на страницу: /login
2025-02-19 06:36:21,140 - INFO - Переход на страницу: /
2025-02-19 06:36:21,140 - ERROR - Future exception was never retrieved
future: <Future finished exception=TypeError("Column.__init__() got an unexpected keyword argument 'bgcolor'")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\home.py", line 26, in _build
    ft.Column([
    ~~~~~~~~~^^
        ft.Text("Welcome to the Mobile Home Page!", size=24, weight=ft.FontWeight.BOLD),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        ft.Text(f"You're using a {device_type}.", size=18),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        ft.ElevatedButton("Go to Mobile Settings", on_click=lambda _: self.page.go("/settings"))
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ], expand=True, spacing=10, bgcolor=ft.Colors.GREY_200, alignment=ft.MainAxisAlignment.START)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Column.__init__() got an unexpected keyword argument 'bgcolor'
2025-02-19 06:36:25,555 - INFO - Session disconnected: _N0e3xPCE4LxF1NtH_7e65f042b7c94ff8759fa7834cb934484151af3d
2025-02-19 06:36:27,456 - INFO - New FletApp: KlC5KPnz
2025-02-19 06:36:27,663 - INFO - New session created (5 total): _ClMaaNUygHqZPlge_d7dbcea8c5375c52c6e52b3f2541725860ece8d5
2025-02-19 06:36:27,663 - INFO - Start session: ClMaaNUygHqZPlge
2025-02-19 06:36:27,743 - INFO - Переход на страницу: /login
2025-02-19 06:36:27,828 - INFO - Переход на страницу: /
2025-02-19 06:36:27,860 - INFO - Переход на страницу: /login
2025-02-19 06:36:27,963 - INFO - Переход на страницу: /
2025-02-19 06:36:29,680 - INFO - Session disconnected: _ClMaaNUygHqZPlge_d7dbcea8c5375c52c6e52b3f2541725860ece8d5
2025-02-19 06:36:31,368 - INFO - New FletApp: JhT1Fsbp
2025-02-19 06:36:31,512 - INFO - Existing session requested: ClMaaNUygHqZPlge
2025-02-19 06:36:31,512 - INFO - Session reconnected: _ClMaaNUygHqZPlge_d7dbcea8c5375c52c6e52b3f2541725860ece8d5
2025-02-19 06:36:31,512 - INFO - Disposing FletApp: KlC5KPnz
2025-02-19 06:36:44,685 - INFO - Session disconnected: _dbhFbJxU6PtMneeQ_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:36:44,686 - INFO - Session disconnected: _ClMaaNUygHqZPlge_d7dbcea8c5375c52c6e52b3f2541725860ece8d5
2025-02-19 06:36:44,792 - INFO - Shutting down Flet App Manager
2025-02-19 06:36:44,792 - INFO - Deleting temp dir: C:\Temp\tmpwiq718fp
2025-02-19 06:36:44,794 - INFO - Flet app has started...
2025-02-19 06:36:44,794 - INFO - Closing Flet web server...
2025-02-19 06:36:51,070 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:36:51,551 - INFO - Starting Flet web server on port 8550...
2025-02-19 06:36:51,552 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 06:36:51,552 - INFO - Use color emoji: False
2025-02-19 06:36:51,552 - INFO - Route URL strategy configured: path
2025-02-19 06:36:51,585 - INFO - Starting up Flet App Manager
2025-02-19 06:36:51,585 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 06:36:51,832 - INFO - New FletApp: tEP8j9be
2025-02-19 06:36:52,154 - INFO - New session created (1 total): _DzZ93R2fnyf3kZRZ_d7dbcea8c5375c52c6e52b3f2541725860ece8d5
2025-02-19 06:36:52,154 - INFO - Start session: DzZ93R2fnyf3kZRZ
2025-02-19 06:36:52,159 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmpsv343_ud
2025-02-19 06:36:52,159 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 06:36:52,159 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 06:36:52,167 - INFO - New FletApp: al9hHFw7
2025-02-19 06:36:52,170 - INFO - Переход на страницу: /login
2025-02-19 06:36:52,193 - INFO - Переход на страницу: /
2025-02-19 06:36:52,195 - INFO - New session created (2 total): _mh6LlG26govuUq4F_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:36:52,196 - INFO - Start session: mh6LlG26govuUq4F
2025-02-19 06:36:52,203 - INFO - Переход на страницу: /login
2025-02-19 06:36:52,225 - INFO - Переход на страницу: /
2025-02-19 06:36:53,433 - INFO - New FletApp: uykNr3wg
2025-02-19 06:36:53,457 - INFO - New session created (3 total): _FY8qGl8XCVdrYpP6_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:36:53,458 - INFO - Start session: FY8qGl8XCVdrYpP6
2025-02-19 06:36:53,573 - INFO - Переход на страницу: /login
2025-02-19 06:36:53,574 - INFO - Session disconnected: _FY8qGl8XCVdrYpP6_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 06:36:54,531 - INFO - Session disconnected: _DzZ93R2fnyf3kZRZ_d7dbcea8c5375c52c6e52b3f2541725860ece8d5
2025-02-19 06:36:56,647 - INFO - New FletApp: 96LREf1E
2025-02-19 06:36:56,746 - INFO - Existing session requested: DzZ93R2fnyf3kZRZ
2025-02-19 06:36:56,746 - INFO - Session reconnected: _DzZ93R2fnyf3kZRZ_d7dbcea8c5375c52c6e52b3f2541725860ece8d5
2025-02-19 06:36:56,746 - INFO - Disposing FletApp: tEP8j9be
2025-02-19 06:36:58,577 - ERROR - Future exception was never retrieved
future: <Future finished exception=TimeoutError("Timeout waiting for invokeMethod clientStorage:get({'key': 'logged_in'}) call")>
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\concurrent\futures\thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\main.py", line 44, in route_change
    self.page.views.append(routes.get(route.route, HomePage)(self.page))
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\base_view.py", line 12, in __init__
    self._build()
    ~~~~~~~~~~~^^
  File "C:\Users\aliaskhan\Documents\Fotrum2\src\pages\login.py", line 7, in _build
    if self.page.client_storage.get("logged_in"):
       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\client_storage.py", line 29, in get
    jv = self.__page._invoke_method(
        "clientStorage:get", {"key": key}, wait_for_result=True
    )
  File "C:\Users\aliaskhan\AppData\Roaming\Python\Python313\site-packages\flet\core\page.py", line 1214, in _invoke_method
    raise TimeoutError(
        f"Timeout waiting for invokeMethod {method_name}({arguments}) call"
    )
TimeoutError: Timeout waiting for invokeMethod clientStorage:get({'key': 'logged_in'}) call
2025-02-19 07:50:02,852 - INFO - Assets path configured: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 07:50:03,509 - INFO - Starting Flet web server on port 8550...
2025-02-19 07:50:03,509 - INFO - Web renderer configured: WebRenderer.CANVAS_KIT
2025-02-19 07:50:03,509 - INFO - Use color emoji: False
2025-02-19 07:50:03,509 - INFO - Route URL strategy configured: path
2025-02-19 07:50:03,590 - INFO - Starting up Flet App Manager
2025-02-19 07:50:03,590 - INFO - App URL: http://127.0.0.1:8550
2025-02-19 07:50:04,114 - INFO - Temp dir created for patched index and manifest: C:\Temp\tmpqfp7ai7n
2025-02-19 07:50:04,115 - INFO - Web root: C:\Program Files\Python313\Lib\site-packages\flet_web\web
2025-02-19 07:50:04,115 - INFO - Assets dir: C:\Users\aliaskhan\Documents\Fotrum2\src\assets
2025-02-19 07:50:05,545 - INFO - New FletApp: 42lZGOxx
2025-02-19 07:50:05,550 - INFO - New session created (1 total): _lb9JtzSyAFUkNdBD_755dd75e7b7626e3138538daab1406e0a0f6f8ae
2025-02-19 07:50:05,550 - INFO - Start session: lb9JtzSyAFUkNdBD
2025-02-19 07:50:05,555 - INFO - Переход на страницу: /login
2025-02-19 07:50:05,558 - INFO - Переход на страницу: /
